2017/11/22 09:32:03 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "favicon.ico".' in D:\zyweb\Yii1\web\CWebApplication.php:286
Stack trace:
#0 D:\zyweb\Yii1\web\CWebApplication.php(141): CWebApplication->runController('favicon.ico')
#1 D:\zyweb\Yii1\base\CApplication.php(180): CWebApplication->processRequest()
#2 D:\zyweb\pcb_logistics\website\home\index.php(38): CApplication->run()
#3 {main}
REQUEST_URI=/favicon.ico
HTTP_REFERER=http://www.wllogistics.com/index/index
---
2017/11/22 09:39:04 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "1".' in D:\zyweb\Yii1\web\CWebApplication.php:286
Stack trace:
#0 D:\zyweb\Yii1\web\CWebApplication.php(141): CWebApplication->runController('1')
#1 D:\zyweb\Yii1\base\CApplication.php(180): CWebApplication->processRequest()
#2 D:\zyweb\pcb_logistics\website\home\index.php(38): CApplication->run()
#3 {main}
REQUEST_URI=/1
HTTP_REFERER=http://www.wllogistics.com/address/list
---
2017/11/22 09:39:08 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "1".' in D:\zyweb\Yii1\web\CWebApplication.php:286
Stack trace:
#0 D:\zyweb\Yii1\web\CWebApplication.php(141): CWebApplication->runController('1')
#1 D:\zyweb\Yii1\base\CApplication.php(180): CWebApplication->processRequest()
#2 D:\zyweb\pcb_logistics\website\home\index.php(38): CApplication->run()
#3 {main}
REQUEST_URI=/1
HTTP_REFERER=http://www.wllogistics.com/address/list
---
2017/11/22 09:39:09 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "1".' in D:\zyweb\Yii1\web\CWebApplication.php:286
Stack trace:
#0 D:\zyweb\Yii1\web\CWebApplication.php(141): CWebApplication->runController('1')
#1 D:\zyweb\Yii1\base\CApplication.php(180): CWebApplication->processRequest()
#2 D:\zyweb\pcb_logistics\website\home\index.php(38): CApplication->run()
#3 {main}
REQUEST_URI=/1
HTTP_REFERER=http://www.wllogistics.com/orders/list
---
2017/11/22 09:58:59 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "1".' in D:\zyweb\Yii1\web\CWebApplication.php:286
Stack trace:
#0 D:\zyweb\Yii1\web\CWebApplication.php(141): CWebApplication->runController('1')
#1 D:\zyweb\Yii1\base\CApplication.php(180): CWebApplication->processRequest()
#2 D:\zyweb\pcb_logistics\website\home\index.php(38): CApplication->run()
#3 {main}
REQUEST_URI=/1
HTTP_REFERER=http://www.wllogistics.com/address/list
---
2017/11/22 10:14:23 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "favicon.ico".' in D:\zyweb\Yii1\web\CWebApplication.php:286
Stack trace:
#0 D:\zyweb\Yii1\web\CWebApplication.php(141): CWebApplication->runController('favicon.ico')
#1 D:\zyweb\Yii1\base\CApplication.php(180): CWebApplication->processRequest()
#2 D:\zyweb\pcb_logistics\website\home\index.php(38): CApplication->run()
#3 {main}
REQUEST_URI=/favicon.ico
HTTP_REFERER=http://www.wllogistics.com/router/routerlist?RouterForm%5Bname%5D=
---
2017/11/22 10:14:23 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "favicon.ico".' in D:\zyweb\Yii1\web\CWebApplication.php:286
Stack trace:
#0 D:\zyweb\Yii1\web\CWebApplication.php(141): CWebApplication->runController('favicon.ico')
#1 D:\zyweb\Yii1\base\CApplication.php(180): CWebApplication->processRequest()
#2 D:\zyweb\pcb_logistics\website\home\index.php(38): CApplication->run()
#3 {main}
REQUEST_URI=/favicon.ico
HTTP_REFERER=http://www.wllogistics.com/router/routerlist?RouterForm%5Bname%5D=
---
2017/11/22 10:14:23 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "favicon.ico".' in D:\zyweb\Yii1\web\CWebApplication.php:286
Stack trace:
#0 D:\zyweb\Yii1\web\CWebApplication.php(141): CWebApplication->runController('favicon.ico')
#1 D:\zyweb\Yii1\base\CApplication.php(180): CWebApplication->processRequest()
#2 D:\zyweb\pcb_logistics\website\home\index.php(38): CApplication->run()
#3 {main}
REQUEST_URI=/favicon.ico
HTTP_REFERER=http://www.wllogistics.com/router/routerlist?RouterForm%5Bname%5D=
---
2017/11/22 10:17:25 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "1".' in D:\zyweb\Yii1\web\CWebApplication.php:286
Stack trace:
#0 D:\zyweb\Yii1\web\CWebApplication.php(141): CWebApplication->runController('1')
#1 D:\zyweb\Yii1\base\CApplication.php(180): CWebApplication->processRequest()
#2 D:\zyweb\pcb_logistics\website\home\index.php(38): CApplication->run()
#3 {main}
REQUEST_URI=/1
HTTP_REFERER=http://www.wllogistics.com/orders/list
---
2017/11/22 10:17:29 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "1".' in D:\zyweb\Yii1\web\CWebApplication.php:286
Stack trace:
#0 D:\zyweb\Yii1\web\CWebApplication.php(141): CWebApplication->runController('1')
#1 D:\zyweb\Yii1\base\CApplication.php(180): CWebApplication->processRequest()
#2 D:\zyweb\pcb_logistics\website\home\index.php(38): CApplication->run()
#3 {main}
REQUEST_URI=/1
HTTP_REFERER=http://www.wllogistics.com/orders/list
---
2017/11/22 10:17:30 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "1".' in D:\zyweb\Yii1\web\CWebApplication.php:286
Stack trace:
#0 D:\zyweb\Yii1\web\CWebApplication.php(141): CWebApplication->runController('1')
#1 D:\zyweb\Yii1\base\CApplication.php(180): CWebApplication->processRequest()
#2 D:\zyweb\pcb_logistics\website\home\index.php(38): CApplication->run()
#3 {main}
REQUEST_URI=/1
HTTP_REFERER=http://www.wllogistics.com/orders/list
---
2017/11/22 10:17:31 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "1".' in D:\zyweb\Yii1\web\CWebApplication.php:286
Stack trace:
#0 D:\zyweb\Yii1\web\CWebApplication.php(141): CWebApplication->runController('1')
#1 D:\zyweb\Yii1\base\CApplication.php(180): CWebApplication->processRequest()
#2 D:\zyweb\pcb_logistics\website\home\index.php(38): CApplication->run()
#3 {main}
REQUEST_URI=/1
HTTP_REFERER=http://www.wllogistics.com/orders/list
---
2017/11/22 10:17:32 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "1".' in D:\zyweb\Yii1\web\CWebApplication.php:286
Stack trace:
#0 D:\zyweb\Yii1\web\CWebApplication.php(141): CWebApplication->runController('1')
#1 D:\zyweb\Yii1\base\CApplication.php(180): CWebApplication->processRequest()
#2 D:\zyweb\pcb_logistics\website\home\index.php(38): CApplication->run()
#3 {main}
REQUEST_URI=/1
HTTP_REFERER=http://www.wllogistics.com/orders/list
---
2017/11/22 10:17:33 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "1".' in D:\zyweb\Yii1\web\CWebApplication.php:286
Stack trace:
#0 D:\zyweb\Yii1\web\CWebApplication.php(141): CWebApplication->runController('1')
#1 D:\zyweb\Yii1\base\CApplication.php(180): CWebApplication->processRequest()
#2 D:\zyweb\pcb_logistics\website\home\index.php(38): CApplication->run()
#3 {main}
REQUEST_URI=/1
HTTP_REFERER=http://www.wllogistics.com/orders/list
---
2017/11/22 10:17:58 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "1".' in D:\zyweb\Yii1\web\CWebApplication.php:286
Stack trace:
#0 D:\zyweb\Yii1\web\CWebApplication.php(141): CWebApplication->runController('1')
#1 D:\zyweb\Yii1\base\CApplication.php(180): CWebApplication->processRequest()
#2 D:\zyweb\pcb_logistics\website\home\index.php(38): CApplication->run()
#3 {main}
REQUEST_URI=/1
HTTP_REFERER=http://www.wllogistics.com/orders/list
---
2017/11/22 10:18:43 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "1".' in D:\zyweb\Yii1\web\CWebApplication.php:286
Stack trace:
#0 D:\zyweb\Yii1\web\CWebApplication.php(141): CWebApplication->runController('1')
#1 D:\zyweb\Yii1\base\CApplication.php(180): CWebApplication->processRequest()
#2 D:\zyweb\pcb_logistics\website\home\index.php(38): CApplication->run()
#3 {main}
REQUEST_URI=/1
HTTP_REFERER=http://www.wllogistics.com/router/routerlist
---
2017/11/22 10:18:44 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "1".' in D:\zyweb\Yii1\web\CWebApplication.php:286
Stack trace:
#0 D:\zyweb\Yii1\web\CWebApplication.php(141): CWebApplication->runController('1')
#1 D:\zyweb\Yii1\base\CApplication.php(180): CWebApplication->processRequest()
#2 D:\zyweb\pcb_logistics\website\home\index.php(38): CApplication->run()
#3 {main}
REQUEST_URI=/1
HTTP_REFERER=http://www.wllogistics.com/router/routerlist
---
2017/11/22 10:18:44 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "1".' in D:\zyweb\Yii1\web\CWebApplication.php:286
Stack trace:
#0 D:\zyweb\Yii1\web\CWebApplication.php(141): CWebApplication->runController('1')
#1 D:\zyweb\Yii1\base\CApplication.php(180): CWebApplication->processRequest()
#2 D:\zyweb\pcb_logistics\website\home\index.php(38): CApplication->run()
#3 {main}
REQUEST_URI=/1
HTTP_REFERER=http://www.wllogistics.com/router/routerlist
---
2017/11/22 10:19:22 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "1".' in D:\zyweb\Yii1\web\CWebApplication.php:286
Stack trace:
#0 D:\zyweb\Yii1\web\CWebApplication.php(141): CWebApplication->runController('1')
#1 D:\zyweb\Yii1\base\CApplication.php(180): CWebApplication->processRequest()
#2 D:\zyweb\pcb_logistics\website\home\index.php(38): CApplication->run()
#3 {main}
REQUEST_URI=/1
HTTP_REFERER=http://www.wllogistics.com/router/routerlist
---
2017/11/22 10:19:24 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "1".' in D:\zyweb\Yii1\web\CWebApplication.php:286
Stack trace:
#0 D:\zyweb\Yii1\web\CWebApplication.php(141): CWebApplication->runController('1')
#1 D:\zyweb\Yii1\base\CApplication.php(180): CWebApplication->processRequest()
#2 D:\zyweb\pcb_logistics\website\home\index.php(38): CApplication->run()
#3 {main}
REQUEST_URI=/1
HTTP_REFERER=http://www.wllogistics.com/orders/list
---
2017/11/22 10:19:25 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "1".' in D:\zyweb\Yii1\web\CWebApplication.php:286
Stack trace:
#0 D:\zyweb\Yii1\web\CWebApplication.php(141): CWebApplication->runController('1')
#1 D:\zyweb\Yii1\base\CApplication.php(180): CWebApplication->processRequest()
#2 D:\zyweb\pcb_logistics\website\home\index.php(38): CApplication->run()
#3 {main}
REQUEST_URI=/1
HTTP_REFERER=http://www.wllogistics.com/orders/list
---
2017/11/22 10:19:25 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "1".' in D:\zyweb\Yii1\web\CWebApplication.php:286
Stack trace:
#0 D:\zyweb\Yii1\web\CWebApplication.php(141): CWebApplication->runController('1')
#1 D:\zyweb\Yii1\base\CApplication.php(180): CWebApplication->processRequest()
#2 D:\zyweb\pcb_logistics\website\home\index.php(38): CApplication->run()
#3 {main}
REQUEST_URI=/1
HTTP_REFERER=http://www.wllogistics.com/orders/list
---
2017/11/22 10:19:25 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "1".' in D:\zyweb\Yii1\web\CWebApplication.php:286
Stack trace:
#0 D:\zyweb\Yii1\web\CWebApplication.php(141): CWebApplication->runController('1')
#1 D:\zyweb\Yii1\base\CApplication.php(180): CWebApplication->processRequest()
#2 D:\zyweb\pcb_logistics\website\home\index.php(38): CApplication->run()
#3 {main}
REQUEST_URI=/1
HTTP_REFERER=http://www.wllogistics.com/orders/list
---
2017/11/22 10:19:26 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "1".' in D:\zyweb\Yii1\web\CWebApplication.php:286
Stack trace:
#0 D:\zyweb\Yii1\web\CWebApplication.php(141): CWebApplication->runController('1')
#1 D:\zyweb\Yii1\base\CApplication.php(180): CWebApplication->processRequest()
#2 D:\zyweb\pcb_logistics\website\home\index.php(38): CApplication->run()
#3 {main}
REQUEST_URI=/1
HTTP_REFERER=http://www.wllogistics.com/orders/list
---
2017/11/22 10:19:26 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "1".' in D:\zyweb\Yii1\web\CWebApplication.php:286
Stack trace:
#0 D:\zyweb\Yii1\web\CWebApplication.php(141): CWebApplication->runController('1')
#1 D:\zyweb\Yii1\base\CApplication.php(180): CWebApplication->processRequest()
#2 D:\zyweb\pcb_logistics\website\home\index.php(38): CApplication->run()
#3 {main}
REQUEST_URI=/1
HTTP_REFERER=http://www.wllogistics.com/orders/list
---
2017/11/22 10:19:26 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "1".' in D:\zyweb\Yii1\web\CWebApplication.php:286
Stack trace:
#0 D:\zyweb\Yii1\web\CWebApplication.php(141): CWebApplication->runController('1')
#1 D:\zyweb\Yii1\base\CApplication.php(180): CWebApplication->processRequest()
#2 D:\zyweb\pcb_logistics\website\home\index.php(38): CApplication->run()
#3 {main}
REQUEST_URI=/1
HTTP_REFERER=http://www.wllogistics.com/orders/list
---
2017/11/22 10:19:26 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "1".' in D:\zyweb\Yii1\web\CWebApplication.php:286
Stack trace:
#0 D:\zyweb\Yii1\web\CWebApplication.php(141): CWebApplication->runController('1')
#1 D:\zyweb\Yii1\base\CApplication.php(180): CWebApplication->processRequest()
#2 D:\zyweb\pcb_logistics\website\home\index.php(38): CApplication->run()
#3 {main}
REQUEST_URI=/1
HTTP_REFERER=http://www.wllogistics.com/orders/list
---
2017/11/22 10:19:28 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "1".' in D:\zyweb\Yii1\web\CWebApplication.php:286
Stack trace:
#0 D:\zyweb\Yii1\web\CWebApplication.php(141): CWebApplication->runController('1')
#1 D:\zyweb\Yii1\base\CApplication.php(180): CWebApplication->processRequest()
#2 D:\zyweb\pcb_logistics\website\home\index.php(38): CApplication->run()
#3 {main}
REQUEST_URI=/1
HTTP_REFERER=http://www.wllogistics.com/orders/list
---
2017/11/22 10:19:48 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "1".' in D:\zyweb\Yii1\web\CWebApplication.php:286
Stack trace:
#0 D:\zyweb\Yii1\web\CWebApplication.php(141): CWebApplication->runController('1')
#1 D:\zyweb\Yii1\base\CApplication.php(180): CWebApplication->processRequest()
#2 D:\zyweb\pcb_logistics\website\home\index.php(38): CApplication->run()
#3 {main}
REQUEST_URI=/1
HTTP_REFERER=http://www.wllogistics.com/router/routerlist
---
2017/11/22 10:19:51 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "1".' in D:\zyweb\Yii1\web\CWebApplication.php:286
Stack trace:
#0 D:\zyweb\Yii1\web\CWebApplication.php(141): CWebApplication->runController('1')
#1 D:\zyweb\Yii1\base\CApplication.php(180): CWebApplication->processRequest()
#2 D:\zyweb\pcb_logistics\website\home\index.php(38): CApplication->run()
#3 {main}
REQUEST_URI=/1
HTTP_REFERER=http://www.wllogistics.com/address/list
---
2017/11/22 10:19:52 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "1".' in D:\zyweb\Yii1\web\CWebApplication.php:286
Stack trace:
#0 D:\zyweb\Yii1\web\CWebApplication.php(141): CWebApplication->runController('1')
#1 D:\zyweb\Yii1\base\CApplication.php(180): CWebApplication->processRequest()
#2 D:\zyweb\pcb_logistics\website\home\index.php(38): CApplication->run()
#3 {main}
REQUEST_URI=/1
HTTP_REFERER=http://www.wllogistics.com/orders/list
---
2017/11/22 10:20:34 [error] [exception.CException] exception 'CException' with message 'Property "CWebUser.getId" is not defined.' in D:\zyweb\Yii1\base\CComponent.php:130
Stack trace:
#0 D:\zyweb\Yii1\web\auth\CWebUser.php(146): CComponent->__get('getId')
#1 D:\zyweb\pcb_logistics\library\model\base\Router.php(53): CWebUser->__get('getId')
#2 D:\zyweb\pcb_logistics\applications\home\views\orders\edit.php(12): Router::getRouterArr()
#3 D:\zyweb\Yii1\web\CBaseController.php(126): require('D:\\zyweb\\pcb_lo...')
#4 D:\zyweb\Yii1\web\CBaseController.php(95): CBaseController->renderInternal('D:\\zyweb\\pcb_lo...', Array, true)
#5 D:\zyweb\Yii1\web\CController.php(869): CBaseController->renderFile('D:\\zyweb\\pcb_lo...', Array, true)
#6 D:\zyweb\Yii1\web\CController.php(782): CController->renderPartial('edit', Array, true)
#7 D:\zyweb\pcb_logistics\applications\home\components\Controller.php(52): CController->render('edit', Array, false)
#8 D:\zyweb\pcb_logistics\applications\home\controllers\OrdersController.php(72): Controller->render('edit', Array)
#9 D:\zyweb\Yii1\web\actions\CInlineAction.php(49): OrdersController->actionEdit()
#10 D:\zyweb\Yii1\web\CController.php(308): CInlineAction->runWithParams(Array)
#11 D:\zyweb\Yii1\web\CController.php(286): CController->runAction(Object(CInlineAction))
#12 D:\zyweb\Yii1\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#13 D:\zyweb\pcb_logistics\applications\home\components\Controller.php(30): CController->run('edit')
#14 D:\zyweb\Yii1\web\CWebApplication.php(282): Controller->run('edit')
#15 D:\zyweb\Yii1\web\CWebApplication.php(141): CWebApplication->runController('orders/edit')
#16 D:\zyweb\Yii1\base\CApplication.php(180): CWebApplication->processRequest()
#17 D:\zyweb\pcb_logistics\website\home\index.php(38): CApplication->run()
#18 {main}
REQUEST_URI=/orders/edit
HTTP_REFERER=http://www.wllogistics.com/orders/edit
---
2017/11/22 10:25:19 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "1".' in D:\zyweb\Yii1\web\CWebApplication.php:286
Stack trace:
#0 D:\zyweb\Yii1\web\CWebApplication.php(141): CWebApplication->runController('1')
#1 D:\zyweb\Yii1\base\CApplication.php(180): CWebApplication->processRequest()
#2 D:\zyweb\pcb_logistics\website\home\index.php(38): CApplication->run()
#3 {main}
REQUEST_URI=/1
HTTP_REFERER=http://www.wllogistics.com/goods/list
---
2017/11/22 10:32:04 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "1".' in D:\zyweb\Yii1\web\CWebApplication.php:286
Stack trace:
#0 D:\zyweb\Yii1\web\CWebApplication.php(141): CWebApplication->runController('1')
#1 D:\zyweb\Yii1\base\CApplication.php(180): CWebApplication->processRequest()
#2 D:\zyweb\pcb_logistics\website\home\index.php(38): CApplication->run()
#3 {main}
REQUEST_URI=/1
HTTP_REFERER=http://www.wllogistics.com/router/routerlist
---
2017/11/22 10:32:06 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "1".' in D:\zyweb\Yii1\web\CWebApplication.php:286
Stack trace:
#0 D:\zyweb\Yii1\web\CWebApplication.php(141): CWebApplication->runController('1')
#1 D:\zyweb\Yii1\base\CApplication.php(180): CWebApplication->processRequest()
#2 D:\zyweb\pcb_logistics\website\home\index.php(38): CApplication->run()
#3 {main}
REQUEST_URI=/1
HTTP_REFERER=http://www.wllogistics.com/router/routerlist
---
2017/11/22 10:32:06 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "1".' in D:\zyweb\Yii1\web\CWebApplication.php:286
Stack trace:
#0 D:\zyweb\Yii1\web\CWebApplication.php(141): CWebApplication->runController('1')
#1 D:\zyweb\Yii1\base\CApplication.php(180): CWebApplication->processRequest()
#2 D:\zyweb\pcb_logistics\website\home\index.php(38): CApplication->run()
#3 {main}
REQUEST_URI=/1
HTTP_REFERER=http://www.wllogistics.com/router/routerlist
---
2017/11/22 10:32:06 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "1".' in D:\zyweb\Yii1\web\CWebApplication.php:286
Stack trace:
#0 D:\zyweb\Yii1\web\CWebApplication.php(141): CWebApplication->runController('1')
#1 D:\zyweb\Yii1\base\CApplication.php(180): CWebApplication->processRequest()
#2 D:\zyweb\pcb_logistics\website\home\index.php(38): CApplication->run()
#3 {main}
REQUEST_URI=/1
HTTP_REFERER=http://www.wllogistics.com/router/routerlist
---
2017/11/22 10:32:08 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "1".' in D:\zyweb\Yii1\web\CWebApplication.php:286
Stack trace:
#0 D:\zyweb\Yii1\web\CWebApplication.php(141): CWebApplication->runController('1')
#1 D:\zyweb\Yii1\base\CApplication.php(180): CWebApplication->processRequest()
#2 D:\zyweb\pcb_logistics\website\home\index.php(38): CApplication->run()
#3 {main}
REQUEST_URI=/1
HTTP_REFERER=http://www.wllogistics.com/router/routerlist
---
2017/11/22 10:32:08 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "favicon.ico".' in D:\zyweb\Yii1\web\CWebApplication.php:286
Stack trace:
#0 D:\zyweb\Yii1\web\CWebApplication.php(141): CWebApplication->runController('favicon.ico')
#1 D:\zyweb\Yii1\base\CApplication.php(180): CWebApplication->processRequest()
#2 D:\zyweb\pcb_logistics\website\home\index.php(38): CApplication->run()
#3 {main}
REQUEST_URI=/favicon.ico
HTTP_REFERER=http://www.wllogistics.com/router/routerlist
---
2017/11/22 10:32:08 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "1".' in D:\zyweb\Yii1\web\CWebApplication.php:286
Stack trace:
#0 D:\zyweb\Yii1\web\CWebApplication.php(141): CWebApplication->runController('1')
#1 D:\zyweb\Yii1\base\CApplication.php(180): CWebApplication->processRequest()
#2 D:\zyweb\pcb_logistics\website\home\index.php(38): CApplication->run()
#3 {main}
REQUEST_URI=/1
HTTP_REFERER=http://www.wllogistics.com/router/routerlist
---
2017/11/22 10:32:08 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "favicon.ico".' in D:\zyweb\Yii1\web\CWebApplication.php:286
Stack trace:
#0 D:\zyweb\Yii1\web\CWebApplication.php(141): CWebApplication->runController('favicon.ico')
#1 D:\zyweb\Yii1\base\CApplication.php(180): CWebApplication->processRequest()
#2 D:\zyweb\pcb_logistics\website\home\index.php(38): CApplication->run()
#3 {main}
REQUEST_URI=/favicon.ico
HTTP_REFERER=http://www.wllogistics.com/router/routerlist
---
2017/11/22 10:32:09 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "1".' in D:\zyweb\Yii1\web\CWebApplication.php:286
Stack trace:
#0 D:\zyweb\Yii1\web\CWebApplication.php(141): CWebApplication->runController('1')
#1 D:\zyweb\Yii1\base\CApplication.php(180): CWebApplication->processRequest()
#2 D:\zyweb\pcb_logistics\website\home\index.php(38): CApplication->run()
#3 {main}
REQUEST_URI=/1
HTTP_REFERER=http://www.wllogistics.com/router/routerlist
---
2017/11/22 10:33:54 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "1".' in D:\zyweb\Yii1\web\CWebApplication.php:286
Stack trace:
#0 D:\zyweb\Yii1\web\CWebApplication.php(141): CWebApplication->runController('1')
#1 D:\zyweb\Yii1\base\CApplication.php(180): CWebApplication->processRequest()
#2 D:\zyweb\pcb_logistics\website\home\index.php(38): CApplication->run()
#3 {main}
REQUEST_URI=/1
HTTP_REFERER=http://www.wllogistics.com/router/routerlist
---
2017/11/22 10:33:57 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "1".' in D:\zyweb\Yii1\web\CWebApplication.php:286
Stack trace:
#0 D:\zyweb\Yii1\web\CWebApplication.php(141): CWebApplication->runController('1')
#1 D:\zyweb\Yii1\base\CApplication.php(180): CWebApplication->processRequest()
#2 D:\zyweb\pcb_logistics\website\home\index.php(38): CApplication->run()
#3 {main}
REQUEST_URI=/1
HTTP_REFERER=http://www.wllogistics.com/router/routerlist
---
2017/11/22 10:39:54 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "1".' in D:\zyweb\Yii1\web\CWebApplication.php:286
Stack trace:
#0 D:\zyweb\Yii1\web\CWebApplication.php(141): CWebApplication->runController('1')
#1 D:\zyweb\Yii1\base\CApplication.php(180): CWebApplication->processRequest()
#2 D:\zyweb\pcb_logistics\website\home\index.php(38): CApplication->run()
#3 {main}
REQUEST_URI=/1
HTTP_REFERER=http://www.wllogistics.com/router/routerlist
---
2017/11/22 10:39:55 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "1".' in D:\zyweb\Yii1\web\CWebApplication.php:286
Stack trace:
#0 D:\zyweb\Yii1\web\CWebApplication.php(141): CWebApplication->runController('1')
#1 D:\zyweb\Yii1\base\CApplication.php(180): CWebApplication->processRequest()
#2 D:\zyweb\pcb_logistics\website\home\index.php(38): CApplication->run()
#3 {main}
REQUEST_URI=/1
HTTP_REFERER=http://www.wllogistics.com/router/routerlist
---
2017/11/22 10:39:55 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "1".' in D:\zyweb\Yii1\web\CWebApplication.php:286
Stack trace:
#0 D:\zyweb\Yii1\web\CWebApplication.php(141): CWebApplication->runController('1')
#1 D:\zyweb\Yii1\base\CApplication.php(180): CWebApplication->processRequest()
#2 D:\zyweb\pcb_logistics\website\home\index.php(38): CApplication->run()
#3 {main}
REQUEST_URI=/1
HTTP_REFERER=http://www.wllogistics.com/router/routerlist
---
2017/11/22 10:39:55 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "1".' in D:\zyweb\Yii1\web\CWebApplication.php:286
Stack trace:
#0 D:\zyweb\Yii1\web\CWebApplication.php(141): CWebApplication->runController('1')
#1 D:\zyweb\Yii1\base\CApplication.php(180): CWebApplication->processRequest()
#2 D:\zyweb\pcb_logistics\website\home\index.php(38): CApplication->run()
#3 {main}
REQUEST_URI=/1
HTTP_REFERER=http://www.wllogistics.com/router/routerlist
---
2017/11/22 10:39:55 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "1".' in D:\zyweb\Yii1\web\CWebApplication.php:286
Stack trace:
#0 D:\zyweb\Yii1\web\CWebApplication.php(141): CWebApplication->runController('1')
#1 D:\zyweb\Yii1\base\CApplication.php(180): CWebApplication->processRequest()
#2 D:\zyweb\pcb_logistics\website\home\index.php(38): CApplication->run()
#3 {main}
REQUEST_URI=/1
HTTP_REFERER=http://www.wllogistics.com/router/routerlist
---
2017/11/22 10:39:55 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "1".' in D:\zyweb\Yii1\web\CWebApplication.php:286
Stack trace:
#0 D:\zyweb\Yii1\web\CWebApplication.php(141): CWebApplication->runController('1')
#1 D:\zyweb\Yii1\base\CApplication.php(180): CWebApplication->processRequest()
#2 D:\zyweb\pcb_logistics\website\home\index.php(38): CApplication->run()
#3 {main}
REQUEST_URI=/1
HTTP_REFERER=http://www.wllogistics.com/router/routerlist
---
2017/11/22 10:39:56 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "1".' in D:\zyweb\Yii1\web\CWebApplication.php:286
Stack trace:
#0 D:\zyweb\Yii1\web\CWebApplication.php(141): CWebApplication->runController('1')
#1 D:\zyweb\Yii1\base\CApplication.php(180): CWebApplication->processRequest()
#2 D:\zyweb\pcb_logistics\website\home\index.php(38): CApplication->run()
#3 {main}
REQUEST_URI=/1
HTTP_REFERER=http://www.wllogistics.com/router/routerlist
---
2017/11/22 10:40:00 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "1".' in D:\zyweb\Yii1\web\CWebApplication.php:286
Stack trace:
#0 D:\zyweb\Yii1\web\CWebApplication.php(141): CWebApplication->runController('1')
#1 D:\zyweb\Yii1\base\CApplication.php(180): CWebApplication->processRequest()
#2 D:\zyweb\pcb_logistics\website\home\index.php(38): CApplication->run()
#3 {main}
REQUEST_URI=/1
HTTP_REFERER=http://www.wllogistics.com/orders/list
---
2017/11/22 10:42:38 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "1".' in D:\zyweb\Yii1\web\CWebApplication.php:286
Stack trace:
#0 D:\zyweb\Yii1\web\CWebApplication.php(141): CWebApplication->runController('1')
#1 D:\zyweb\Yii1\base\CApplication.php(180): CWebApplication->processRequest()
#2 D:\zyweb\pcb_logistics\website\home\index.php(38): CApplication->run()
#3 {main}
REQUEST_URI=/1
HTTP_REFERER=http://www.wllogistics.com/goods/list
---
2017/11/22 10:42:39 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "1".' in D:\zyweb\Yii1\web\CWebApplication.php:286
Stack trace:
#0 D:\zyweb\Yii1\web\CWebApplication.php(141): CWebApplication->runController('1')
#1 D:\zyweb\Yii1\base\CApplication.php(180): CWebApplication->processRequest()
#2 D:\zyweb\pcb_logistics\website\home\index.php(38): CApplication->run()
#3 {main}
REQUEST_URI=/1
HTTP_REFERER=http://www.wllogistics.com/orders/list
---
2017/11/22 10:52:36 [error] [exception.Exception] exception 'Exception' with message '终点 is too long (maximum is 20 characters).' in D:\zyweb\pcb_logistics\applications\home\service\OrdersService.php:199
Stack trace:
#0 D:\zyweb\pcb_logistics\applications\home\controllers\OrdersController.php(49): OrdersService->addOrder(Array, Object(OrdersForm))
#1 D:\zyweb\Yii1\web\actions\CInlineAction.php(49): OrdersController->actionEdit()
#2 D:\zyweb\Yii1\web\CController.php(308): CInlineAction->runWithParams(Array)
#3 D:\zyweb\Yii1\web\CController.php(286): CController->runAction(Object(CInlineAction))
#4 D:\zyweb\Yii1\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#5 D:\zyweb\pcb_logistics\applications\home\components\Controller.php(30): CController->run('edit')
#6 D:\zyweb\Yii1\web\CWebApplication.php(282): Controller->run('edit')
#7 D:\zyweb\Yii1\web\CWebApplication.php(141): CWebApplication->runController('orders/edit')
#8 D:\zyweb\Yii1\base\CApplication.php(180): CWebApplication->processRequest()
#9 D:\zyweb\pcb_logistics\website\home\index.php(38): CApplication->run()
#10 {main}
REQUEST_URI=/orders/edit
HTTP_REFERER=http://www.wllogistics.com/orders/edit
---
2017/11/22 10:56:54 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "favicon.ico".' in D:\zyweb\Yii1\web\CWebApplication.php:286
Stack trace:
#0 D:\zyweb\Yii1\web\CWebApplication.php(141): CWebApplication->runController('favicon.ico')
#1 D:\zyweb\Yii1\base\CApplication.php(180): CWebApplication->processRequest()
#2 D:\zyweb\pcb_logistics\website\admin\index.php(32): CApplication->run()
#3 {main}
REQUEST_URI=/favicon.ico
HTTP_REFERER=http://admin.wllogistics.com/gii/default/login
---
2017/11/22 10:57:04 [warning] [application] Failed to set unsafe attribute "baseFormClass" of "GiixAllCode".
in D:\zyweb\pcb_logistics\website\admin\index.php (32)
2017/11/22 10:57:04 [warning] [application] Failed to set unsafe attribute "formPath" of "GiixAllCode".
in D:\zyweb\pcb_logistics\website\admin\index.php (32)
2017/11/22 10:57:05 [warning] [application] Failed to set unsafe attribute "baseFormClass" of "GiixAllCode".
in D:\zyweb\pcb_logistics\website\admin\index.php (32)
2017/11/22 10:57:05 [warning] [application] Failed to set unsafe attribute "formPath" of "GiixAllCode".
in D:\zyweb\pcb_logistics\website\admin\index.php (32)
2017/11/22 10:58:18 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "1".' in D:\zyweb\Yii1\web\CWebApplication.php:286
Stack trace:
#0 D:\zyweb\Yii1\web\CWebApplication.php(141): CWebApplication->runController('1')
#1 D:\zyweb\Yii1\base\CApplication.php(180): CWebApplication->processRequest()
#2 D:\zyweb\pcb_logistics\website\home\index.php(38): CApplication->run()
#3 {main}
REQUEST_URI=/1
HTTP_REFERER=http://www.wllogistics.com/Orders/list
---
2017/11/22 10:58:30 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "1".' in D:\zyweb\Yii1\web\CWebApplication.php:286
Stack trace:
#0 D:\zyweb\Yii1\web\CWebApplication.php(141): CWebApplication->runController('1')
#1 D:\zyweb\Yii1\base\CApplication.php(180): CWebApplication->processRequest()
#2 D:\zyweb\pcb_logistics\website\home\index.php(38): CApplication->run()
#3 {main}
REQUEST_URI=/1
HTTP_REFERER=http://www.wllogistics.com/orders/list
---
2017/11/22 11:13:44 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "1".' in D:\zyweb\Yii1\web\CWebApplication.php:286
Stack trace:
#0 D:\zyweb\Yii1\web\CWebApplication.php(141): CWebApplication->runController('1')
#1 D:\zyweb\Yii1\base\CApplication.php(180): CWebApplication->processRequest()
#2 D:\zyweb\pcb_logistics\website\home\index.php(38): CApplication->run()
#3 {main}
REQUEST_URI=/1
HTTP_REFERER=http://www.wllogistics.com/orders/list
---
2017/11/22 11:14:28 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "1".' in D:\zyweb\Yii1\web\CWebApplication.php:286
Stack trace:
#0 D:\zyweb\Yii1\web\CWebApplication.php(141): CWebApplication->runController('1')
#1 D:\zyweb\Yii1\base\CApplication.php(180): CWebApplication->processRequest()
#2 D:\zyweb\pcb_logistics\website\home\index.php(38): CApplication->run()
#3 {main}
REQUEST_URI=/1
HTTP_REFERER=http://www.wllogistics.com/Orders/list
---
2017/11/22 11:36:37 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "1".' in D:\zyweb\Yii1\web\CWebApplication.php:286
Stack trace:
#0 D:\zyweb\Yii1\web\CWebApplication.php(141): CWebApplication->runController('1')
#1 D:\zyweb\Yii1\base\CApplication.php(180): CWebApplication->processRequest()
#2 D:\zyweb\pcb_logistics\website\home\index.php(38): CApplication->run()
#3 {main}
REQUEST_URI=/1
HTTP_REFERER=http://www.wllogistics.com/orders/list
---
2017/11/22 11:38:19 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "1".' in D:\zyweb\Yii1\web\CWebApplication.php:286
Stack trace:
#0 D:\zyweb\Yii1\web\CWebApplication.php(141): CWebApplication->runController('1')
#1 D:\zyweb\Yii1\base\CApplication.php(180): CWebApplication->processRequest()
#2 D:\zyweb\pcb_logistics\website\home\index.php(38): CApplication->run()
#3 {main}
REQUEST_URI=/1
HTTP_REFERER=http://www.wllogistics.com/orders/list
---
2017/11/22 11:38:20 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "1".' in D:\zyweb\Yii1\web\CWebApplication.php:286
Stack trace:
#0 D:\zyweb\Yii1\web\CWebApplication.php(141): CWebApplication->runController('1')
#1 D:\zyweb\Yii1\base\CApplication.php(180): CWebApplication->processRequest()
#2 D:\zyweb\pcb_logistics\website\home\index.php(38): CApplication->run()
#3 {main}
REQUEST_URI=/1
HTTP_REFERER=http://www.wllogistics.com/orders/list
---
2017/11/22 11:38:20 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "1".' in D:\zyweb\Yii1\web\CWebApplication.php:286
Stack trace:
#0 D:\zyweb\Yii1\web\CWebApplication.php(141): CWebApplication->runController('1')
#1 D:\zyweb\Yii1\base\CApplication.php(180): CWebApplication->processRequest()
#2 D:\zyweb\pcb_logistics\website\home\index.php(38): CApplication->run()
#3 {main}
REQUEST_URI=/1
HTTP_REFERER=http://www.wllogistics.com/orders/list
---
2017/11/22 11:38:20 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "1".' in D:\zyweb\Yii1\web\CWebApplication.php:286
Stack trace:
#0 D:\zyweb\Yii1\web\CWebApplication.php(141): CWebApplication->runController('1')
#1 D:\zyweb\Yii1\base\CApplication.php(180): CWebApplication->processRequest()
#2 D:\zyweb\pcb_logistics\website\home\index.php(38): CApplication->run()
#3 {main}
REQUEST_URI=/1
HTTP_REFERER=http://www.wllogistics.com/orders/list
---
2017/11/22 11:38:20 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "1".' in D:\zyweb\Yii1\web\CWebApplication.php:286
Stack trace:
#0 D:\zyweb\Yii1\web\CWebApplication.php(141): CWebApplication->runController('1')
#1 D:\zyweb\Yii1\base\CApplication.php(180): CWebApplication->processRequest()
#2 D:\zyweb\pcb_logistics\website\home\index.php(38): CApplication->run()
#3 {main}
REQUEST_URI=/1
HTTP_REFERER=http://www.wllogistics.com/orders/list
---
2017/11/22 11:38:20 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "1".' in D:\zyweb\Yii1\web\CWebApplication.php:286
Stack trace:
#0 D:\zyweb\Yii1\web\CWebApplication.php(141): CWebApplication->runController('1')
#1 D:\zyweb\Yii1\base\CApplication.php(180): CWebApplication->processRequest()
#2 D:\zyweb\pcb_logistics\website\home\index.php(38): CApplication->run()
#3 {main}
REQUEST_URI=/1
HTTP_REFERER=http://www.wllogistics.com/orders/list
---
2017/11/22 13:28:16 [error] [system.db.CDbCommand] CDbCommand::fetch() failed: SQLSTATE[42000]: Syntax error or access violation: 1064 You have an error in your SQL syntax; check the manual that corresponds to your MySQL server version for the right syntax to use near '123123123213 FROM `Orders` `t` WHERE ((userId="6") AND (state="1")) AND (endTime' at line 1. The SQL statement executed was: SELECT sum(sumWeight)123123123213 FROM `Orders` `t` WHERE ((userId="6") AND (state="1")) AND (endTime BETWEEN "2017-11-20 00:00:00" AND "2017-11-26 23:59:59").
in D:\zyweb\pcb_logistics\library\model\BaseModel.php (324)
in D:\zyweb\pcb_logistics\library\model\BaseModel.php (227)
in D:\zyweb\pcb_logistics\library\model\base\Orders.php (275)
2017/11/22 13:28:16 [error] [exception.CDbException] exception 'CDbException' with message 'CDbCommand failed to execute the SQL statement: SQLSTATE[42000]: Syntax error or access violation: 1064 You have an error in your SQL syntax; check the manual that corresponds to your MySQL server version for the right syntax to use near '123123123213 FROM `Orders` `t` WHERE ((userId="6") AND (state="1")) AND (endTime' at line 1. The SQL statement executed was: SELECT sum(sumWeight)123123123213 FROM `Orders` `t` WHERE ((userId="6") AND (state="1")) AND (endTime BETWEEN "2017-11-20 00:00:00" AND "2017-11-26 23:59:59")' in D:\zyweb\Yii1\db\CDbCommand.php:559
Stack trace:
#0 D:\zyweb\Yii1\db\CDbCommand.php(426): CDbCommand->queryInternal('fetch', Array, Array)
#1 D:\zyweb\pcb_logistics\library\model\BaseModel.php(324): CDbCommand->queryRow()
#2 D:\zyweb\pcb_logistics\library\model\BaseModel.php(227): BaseModel->getData('SELECT sum(sumW...', 'queryRow')
#3 D:\zyweb\pcb_logistics\library\model\base\Orders.php(275): BaseModel->query(Object(CDbCriteria), 'queryRow')
#4 D:\zyweb\pcb_logistics\applications\home\controllers\ManageController.php(61): Orders->getSumWeight('2')
#5 D:\zyweb\Yii1\web\actions\CInlineAction.php(49): ManageController->actionDataList()
#6 D:\zyweb\Yii1\web\CController.php(308): CInlineAction->runWithParams(Array)
#7 D:\zyweb\Yii1\web\CController.php(286): CController->runAction(Object(CInlineAction))
#8 D:\zyweb\Yii1\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#9 D:\zyweb\pcb_logistics\applications\home\components\Controller.php(30): CController->run('DataList')
#10 D:\zyweb\Yii1\web\CWebApplication.php(282): Controller->run('DataList')
#11 D:\zyweb\Yii1\web\CWebApplication.php(141): CWebApplication->runController('manage/DataList')
#12 D:\zyweb\Yii1\base\CApplication.php(180): CWebApplication->processRequest()
#13 D:\zyweb\pcb_logistics\website\home\index.php(38): CApplication->run()
#14 {main}
REQUEST_URI=/manage/DataList?type=2
---
2017/11/22 13:29:33 [error] [system.db.CDbCommand] CDbCommand::fetch() failed: SQLSTATE[42000]: Syntax error or access violation: 1064 You have an error in your SQL syntax; check the manual that corresponds to your MySQL server version for the right syntax to use near '111 FROM `Orders` `t` WHERE ((userId="6") AND (state="1")) AND (endTime BETWEEN ' at line 1. The SQL statement executed was: SELECT sum(sumWeight)111 FROM `Orders` `t` WHERE ((userId="6") AND (state="1")) AND (endTime BETWEEN "2017-11-22 00:00:00" AND "2017-11-22 23:59:59").
in D:\zyweb\pcb_logistics\library\model\BaseModel.php (324)
in D:\zyweb\pcb_logistics\library\model\BaseModel.php (227)
in D:\zyweb\pcb_logistics\library\model\base\Orders.php (275)
2017/11/22 13:29:33 [error] [exception.CDbException] exception 'CDbException' with message 'CDbCommand failed to execute the SQL statement: SQLSTATE[42000]: Syntax error or access violation: 1064 You have an error in your SQL syntax; check the manual that corresponds to your MySQL server version for the right syntax to use near '111 FROM `Orders` `t` WHERE ((userId="6") AND (state="1")) AND (endTime BETWEEN ' at line 1. The SQL statement executed was: SELECT sum(sumWeight)111 FROM `Orders` `t` WHERE ((userId="6") AND (state="1")) AND (endTime BETWEEN "2017-11-22 00:00:00" AND "2017-11-22 23:59:59")' in D:\zyweb\Yii1\db\CDbCommand.php:559
Stack trace:
#0 D:\zyweb\Yii1\db\CDbCommand.php(426): CDbCommand->queryInternal('fetch', Array, Array)
#1 D:\zyweb\pcb_logistics\library\model\BaseModel.php(324): CDbCommand->queryRow()
#2 D:\zyweb\pcb_logistics\library\model\BaseModel.php(227): BaseModel->getData('SELECT sum(sumW...', 'queryRow')
#3 D:\zyweb\pcb_logistics\library\model\base\Orders.php(275): BaseModel->query(Object(CDbCriteria), 'queryRow')
#4 D:\zyweb\pcb_logistics\applications\home\controllers\ManageController.php(61): Orders->getSumWeight('1')
#5 D:\zyweb\Yii1\web\actions\CInlineAction.php(49): ManageController->actionDataList()
#6 D:\zyweb\Yii1\web\CController.php(308): CInlineAction->runWithParams(Array)
#7 D:\zyweb\Yii1\web\CController.php(286): CController->runAction(Object(CInlineAction))
#8 D:\zyweb\Yii1\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#9 D:\zyweb\pcb_logistics\applications\home\components\Controller.php(30): CController->run('DataList')
#10 D:\zyweb\Yii1\web\CWebApplication.php(282): Controller->run('DataList')
#11 D:\zyweb\Yii1\web\CWebApplication.php(141): CWebApplication->runController('manage/DataList')
#12 D:\zyweb\Yii1\base\CApplication.php(180): CWebApplication->processRequest()
#13 D:\zyweb\pcb_logistics\website\home\index.php(38): CApplication->run()
#14 {main}
REQUEST_URI=/manage/DataList?type=1
---
2017/11/22 13:30:04 [error] [system.db.CDbCommand] CDbCommand::fetch() failed: SQLSTATE[42000]: Syntax error or access violation: 1064 You have an error in your SQL syntax; check the manual that corresponds to your MySQL server version for the right syntax to use near '111 FROM `Orders` `t` WHERE ((userId="6") AND (state="1")) AND (endTime BETWEEN ' at line 1. The SQL statement executed was: SELECT sum(sumWeight)111 FROM `Orders` `t` WHERE ((userId="6") AND (state="1")) AND (endTime BETWEEN "2017-11-22 00:00:00" AND "2017-11-22 23:59:59").
in D:\zyweb\pcb_logistics\library\model\BaseModel.php (324)
in D:\zyweb\pcb_logistics\library\model\BaseModel.php (227)
in D:\zyweb\pcb_logistics\library\model\base\Orders.php (275)
2017/11/22 13:30:04 [error] [exception.CDbException] exception 'CDbException' with message 'CDbCommand failed to execute the SQL statement: SQLSTATE[42000]: Syntax error or access violation: 1064 You have an error in your SQL syntax; check the manual that corresponds to your MySQL server version for the right syntax to use near '111 FROM `Orders` `t` WHERE ((userId="6") AND (state="1")) AND (endTime BETWEEN ' at line 1. The SQL statement executed was: SELECT sum(sumWeight)111 FROM `Orders` `t` WHERE ((userId="6") AND (state="1")) AND (endTime BETWEEN "2017-11-22 00:00:00" AND "2017-11-22 23:59:59")' in D:\zyweb\Yii1\db\CDbCommand.php:559
Stack trace:
#0 D:\zyweb\Yii1\db\CDbCommand.php(426): CDbCommand->queryInternal('fetch', Array, Array)
#1 D:\zyweb\pcb_logistics\library\model\BaseModel.php(324): CDbCommand->queryRow()
#2 D:\zyweb\pcb_logistics\library\model\BaseModel.php(227): BaseModel->getData('SELECT sum(sumW...', 'queryRow')
#3 D:\zyweb\pcb_logistics\library\model\base\Orders.php(275): BaseModel->query(Object(CDbCriteria), 'queryRow')
#4 D:\zyweb\pcb_logistics\applications\home\controllers\ManageController.php(61): Orders->getSumWeight('1')
#5 D:\zyweb\Yii1\web\actions\CInlineAction.php(49): ManageController->actionDataList()
#6 D:\zyweb\Yii1\web\CController.php(308): CInlineAction->runWithParams(Array)
#7 D:\zyweb\Yii1\web\CController.php(286): CController->runAction(Object(CInlineAction))
#8 D:\zyweb\Yii1\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#9 D:\zyweb\pcb_logistics\applications\home\components\Controller.php(30): CController->run('DataList')
#10 D:\zyweb\Yii1\web\CWebApplication.php(282): Controller->run('DataList')
#11 D:\zyweb\Yii1\web\CWebApplication.php(141): CWebApplication->runController('manage/DataList')
#12 D:\zyweb\Yii1\base\CApplication.php(180): CWebApplication->processRequest()
#13 D:\zyweb\pcb_logistics\website\home\index.php(38): CApplication->run()
#14 {main}
REQUEST_URI=/manage/DataList?type=1
---
2017/11/22 13:39:09 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'The system is unable to find the requested action "index".' in D:\zyweb\Yii1\web\CController.php:483
Stack trace:
#0 D:\zyweb\Yii1\web\CController.php(270): CController->missingAction('index')
#1 D:\zyweb\pcb_logistics\applications\home\components\Controller.php(30): CController->run('index')
#2 D:\zyweb\Yii1\web\CWebApplication.php(282): Controller->run('index')
#3 D:\zyweb\Yii1\web\CWebApplication.php(141): CWebApplication->runController('user/index')
#4 D:\zyweb\Yii1\base\CApplication.php(180): CWebApplication->processRequest()
#5 D:\zyweb\pcb_logistics\website\home\index.php(38): CApplication->run()
#6 {main}
REQUEST_URI=/user/index
HTTP_REFERER=http://www.wllogistics.com/manage/index
---
2017/11/22 13:41:52 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "1".' in D:\zyweb\Yii1\web\CWebApplication.php:286
Stack trace:
#0 D:\zyweb\Yii1\web\CWebApplication.php(141): CWebApplication->runController('1')
#1 D:\zyweb\Yii1\base\CApplication.php(180): CWebApplication->processRequest()
#2 D:\zyweb\pcb_logistics\website\home\index.php(38): CApplication->run()
#3 {main}
REQUEST_URI=/1
HTTP_REFERER=http://www.wllogistics.com/orders/list
---
2017/11/22 13:41:54 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "1".' in D:\zyweb\Yii1\web\CWebApplication.php:286
Stack trace:
#0 D:\zyweb\Yii1\web\CWebApplication.php(141): CWebApplication->runController('1')
#1 D:\zyweb\Yii1\base\CApplication.php(180): CWebApplication->processRequest()
#2 D:\zyweb\pcb_logistics\website\home\index.php(38): CApplication->run()
#3 {main}
REQUEST_URI=/1
HTTP_REFERER=http://www.wllogistics.com/orders/list
---
2017/11/22 13:47:17 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "1".' in D:\zyweb\Yii1\web\CWebApplication.php:286
Stack trace:
#0 D:\zyweb\Yii1\web\CWebApplication.php(141): CWebApplication->runController('1')
#1 D:\zyweb\Yii1\base\CApplication.php(180): CWebApplication->processRequest()
#2 D:\zyweb\pcb_logistics\website\home\index.php(38): CApplication->run()
#3 {main}
REQUEST_URI=/1
HTTP_REFERER=http://www.wllogistics.com/orders/list
---
2017/11/22 13:47:29 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "1".' in D:\zyweb\Yii1\web\CWebApplication.php:286
Stack trace:
#0 D:\zyweb\Yii1\web\CWebApplication.php(141): CWebApplication->runController('1')
#1 D:\zyweb\Yii1\base\CApplication.php(180): CWebApplication->processRequest()
#2 D:\zyweb\pcb_logistics\website\home\index.php(38): CApplication->run()
#3 {main}
REQUEST_URI=/1
HTTP_REFERER=http://www.wllogistics.com/orders/list
---
2017/11/22 13:47:30 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "1".' in D:\zyweb\Yii1\web\CWebApplication.php:286
Stack trace:
#0 D:\zyweb\Yii1\web\CWebApplication.php(141): CWebApplication->runController('1')
#1 D:\zyweb\Yii1\base\CApplication.php(180): CWebApplication->processRequest()
#2 D:\zyweb\pcb_logistics\website\home\index.php(38): CApplication->run()
#3 {main}
REQUEST_URI=/1
HTTP_REFERER=http://www.wllogistics.com/orders/list
---
2017/11/22 13:50:46 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "favicon.ico".' in D:\zyweb\Yii1\web\CWebApplication.php:286
Stack trace:
#0 D:\zyweb\Yii1\web\CWebApplication.php(141): CWebApplication->runController('favicon.ico')
#1 D:\zyweb\Yii1\base\CApplication.php(180): CWebApplication->processRequest()
#2 D:\zyweb\pcb_logistics\website\home\index.php(38): CApplication->run()
#3 {main}
REQUEST_URI=/favicon.ico
HTTP_REFERER=http://www.wllogistics.com/orders/edit
---
2017/11/22 13:53:40 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "1".' in D:\zyweb\Yii1\web\CWebApplication.php:286
Stack trace:
#0 D:\zyweb\Yii1\web\CWebApplication.php(141): CWebApplication->runController('1')
#1 D:\zyweb\Yii1\base\CApplication.php(180): CWebApplication->processRequest()
#2 D:\zyweb\pcb_logistics\website\home\index.php(38): CApplication->run()
#3 {main}
REQUEST_URI=/1
HTTP_REFERER=http://www.wllogistics.com/orders/list
---
2017/11/22 13:53:42 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "1".' in D:\zyweb\Yii1\web\CWebApplication.php:286
Stack trace:
#0 D:\zyweb\Yii1\web\CWebApplication.php(141): CWebApplication->runController('1')
#1 D:\zyweb\Yii1\base\CApplication.php(180): CWebApplication->processRequest()
#2 D:\zyweb\pcb_logistics\website\home\index.php(38): CApplication->run()
#3 {main}
REQUEST_URI=/1
HTTP_REFERER=http://www.wllogistics.com/goods/list
---
2017/11/22 13:53:44 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "1".' in D:\zyweb\Yii1\web\CWebApplication.php:286
Stack trace:
#0 D:\zyweb\Yii1\web\CWebApplication.php(141): CWebApplication->runController('1')
#1 D:\zyweb\Yii1\base\CApplication.php(180): CWebApplication->processRequest()
#2 D:\zyweb\pcb_logistics\website\home\index.php(38): CApplication->run()
#3 {main}
REQUEST_URI=/1
HTTP_REFERER=http://www.wllogistics.com/orders/list
---
2017/11/22 13:54:55 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "1".' in D:\zyweb\Yii1\web\CWebApplication.php:286
Stack trace:
#0 D:\zyweb\Yii1\web\CWebApplication.php(141): CWebApplication->runController('1')
#1 D:\zyweb\Yii1\base\CApplication.php(180): CWebApplication->processRequest()
#2 D:\zyweb\pcb_logistics\website\home\index.php(38): CApplication->run()
#3 {main}
REQUEST_URI=/1
HTTP_REFERER=http://www.wllogistics.com/orders/list
---
2017/11/22 13:54:56 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "1".' in D:\zyweb\Yii1\web\CWebApplication.php:286
Stack trace:
#0 D:\zyweb\Yii1\web\CWebApplication.php(141): CWebApplication->runController('1')
#1 D:\zyweb\Yii1\base\CApplication.php(180): CWebApplication->processRequest()
#2 D:\zyweb\pcb_logistics\website\home\index.php(38): CApplication->run()
#3 {main}
REQUEST_URI=/1
HTTP_REFERER=http://www.wllogistics.com/orders/list
---
2017/11/22 13:54:57 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "1".' in D:\zyweb\Yii1\web\CWebApplication.php:286
Stack trace:
#0 D:\zyweb\Yii1\web\CWebApplication.php(141): CWebApplication->runController('1')
#1 D:\zyweb\Yii1\base\CApplication.php(180): CWebApplication->processRequest()
#2 D:\zyweb\pcb_logistics\website\home\index.php(38): CApplication->run()
#3 {main}
REQUEST_URI=/1
HTTP_REFERER=http://www.wllogistics.com/orders/list
---
2017/11/22 13:54:58 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "1".' in D:\zyweb\Yii1\web\CWebApplication.php:286
Stack trace:
#0 D:\zyweb\Yii1\web\CWebApplication.php(141): CWebApplication->runController('1')
#1 D:\zyweb\Yii1\base\CApplication.php(180): CWebApplication->processRequest()
#2 D:\zyweb\pcb_logistics\website\home\index.php(38): CApplication->run()
#3 {main}
REQUEST_URI=/1
HTTP_REFERER=http://www.wllogistics.com/orders/list
---
2017/11/22 13:54:59 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "1".' in D:\zyweb\Yii1\web\CWebApplication.php:286
Stack trace:
#0 D:\zyweb\Yii1\web\CWebApplication.php(141): CWebApplication->runController('1')
#1 D:\zyweb\Yii1\base\CApplication.php(180): CWebApplication->processRequest()
#2 D:\zyweb\pcb_logistics\website\home\index.php(38): CApplication->run()
#3 {main}
REQUEST_URI=/1
HTTP_REFERER=http://www.wllogistics.com/orders/list
---
2017/11/22 13:55:00 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "1".' in D:\zyweb\Yii1\web\CWebApplication.php:286
Stack trace:
#0 D:\zyweb\Yii1\web\CWebApplication.php(141): CWebApplication->runController('1')
#1 D:\zyweb\Yii1\base\CApplication.php(180): CWebApplication->processRequest()
#2 D:\zyweb\pcb_logistics\website\home\index.php(38): CApplication->run()
#3 {main}
REQUEST_URI=/1
HTTP_REFERER=http://www.wllogistics.com/orders/list
---
2017/11/22 13:55:02 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "1".' in D:\zyweb\Yii1\web\CWebApplication.php:286
Stack trace:
#0 D:\zyweb\Yii1\web\CWebApplication.php(141): CWebApplication->runController('1')
#1 D:\zyweb\Yii1\base\CApplication.php(180): CWebApplication->processRequest()
#2 D:\zyweb\pcb_logistics\website\home\index.php(38): CApplication->run()
#3 {main}
REQUEST_URI=/1
HTTP_REFERER=http://www.wllogistics.com/orders/list
---
2017/11/22 13:55:06 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "1".' in D:\zyweb\Yii1\web\CWebApplication.php:286
Stack trace:
#0 D:\zyweb\Yii1\web\CWebApplication.php(141): CWebApplication->runController('1')
#1 D:\zyweb\Yii1\base\CApplication.php(180): CWebApplication->processRequest()
#2 D:\zyweb\pcb_logistics\website\home\index.php(38): CApplication->run()
#3 {main}
REQUEST_URI=/1
HTTP_REFERER=http://www.wllogistics.com/orders/list
---
2017/11/22 13:59:45 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "favicon.ico".' in D:\zyweb\Yii1\web\CWebApplication.php:286
Stack trace:
#0 D:\zyweb\Yii1\web\CWebApplication.php(141): CWebApplication->runController('favicon.ico')
#1 D:\zyweb\Yii1\base\CApplication.php(180): CWebApplication->processRequest()
#2 D:\zyweb\pcb_logistics\website\home\index.php(38): CApplication->run()
#3 {main}
REQUEST_URI=/favicon.ico
HTTP_REFERER=http://www.wllogistics.com/index/index
---
2017/11/22 14:00:13 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "1".' in D:\zyweb\Yii1\web\CWebApplication.php:286
Stack trace:
#0 D:\zyweb\Yii1\web\CWebApplication.php(141): CWebApplication->runController('1')
#1 D:\zyweb\Yii1\base\CApplication.php(180): CWebApplication->processRequest()
#2 D:\zyweb\pcb_logistics\website\home\index.php(38): CApplication->run()
#3 {main}
REQUEST_URI=/1
HTTP_REFERER=http://www.wllogistics.com/orders/list
---
2017/11/22 14:01:36 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "1".' in D:\zyweb\Yii1\web\CWebApplication.php:286
Stack trace:
#0 D:\zyweb\Yii1\web\CWebApplication.php(141): CWebApplication->runController('1')
#1 D:\zyweb\Yii1\base\CApplication.php(180): CWebApplication->processRequest()
#2 D:\zyweb\pcb_logistics\website\home\index.php(38): CApplication->run()
#3 {main}
REQUEST_URI=/1
HTTP_REFERER=http://www.wllogistics.com/orders/list
---
2017/11/22 14:01:45 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "1".' in D:\zyweb\Yii1\web\CWebApplication.php:286
Stack trace:
#0 D:\zyweb\Yii1\web\CWebApplication.php(141): CWebApplication->runController('1')
#1 D:\zyweb\Yii1\base\CApplication.php(180): CWebApplication->processRequest()
#2 D:\zyweb\pcb_logistics\website\home\index.php(38): CApplication->run()
#3 {main}
REQUEST_URI=/1
HTTP_REFERER=http://www.wllogistics.com/orders/list
---
2017/11/22 14:01:45 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "1".' in D:\zyweb\Yii1\web\CWebApplication.php:286
Stack trace:
#0 D:\zyweb\Yii1\web\CWebApplication.php(141): CWebApplication->runController('1')
#1 D:\zyweb\Yii1\base\CApplication.php(180): CWebApplication->processRequest()
#2 D:\zyweb\pcb_logistics\website\home\index.php(38): CApplication->run()
#3 {main}
REQUEST_URI=/1
HTTP_REFERER=http://www.wllogistics.com/orders/list
---
2017/11/22 14:01:45 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "1".' in D:\zyweb\Yii1\web\CWebApplication.php:286
Stack trace:
#0 D:\zyweb\Yii1\web\CWebApplication.php(141): CWebApplication->runController('1')
#1 D:\zyweb\Yii1\base\CApplication.php(180): CWebApplication->processRequest()
#2 D:\zyweb\pcb_logistics\website\home\index.php(38): CApplication->run()
#3 {main}
REQUEST_URI=/1
HTTP_REFERER=http://www.wllogistics.com/orders/list
---
2017/11/22 14:01:45 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "1".' in D:\zyweb\Yii1\web\CWebApplication.php:286
Stack trace:
#0 D:\zyweb\Yii1\web\CWebApplication.php(141): CWebApplication->runController('1')
#1 D:\zyweb\Yii1\base\CApplication.php(180): CWebApplication->processRequest()
#2 D:\zyweb\pcb_logistics\website\home\index.php(38): CApplication->run()
#3 {main}
REQUEST_URI=/1
HTTP_REFERER=http://www.wllogistics.com/orders/list
---
2017/11/22 14:01:46 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "1".' in D:\zyweb\Yii1\web\CWebApplication.php:286
Stack trace:
#0 D:\zyweb\Yii1\web\CWebApplication.php(141): CWebApplication->runController('1')
#1 D:\zyweb\Yii1\base\CApplication.php(180): CWebApplication->processRequest()
#2 D:\zyweb\pcb_logistics\website\home\index.php(38): CApplication->run()
#3 {main}
REQUEST_URI=/1
HTTP_REFERER=http://www.wllogistics.com/orders/list
---
2017/11/22 14:01:46 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "1".' in D:\zyweb\Yii1\web\CWebApplication.php:286
Stack trace:
#0 D:\zyweb\Yii1\web\CWebApplication.php(141): CWebApplication->runController('1')
#1 D:\zyweb\Yii1\base\CApplication.php(180): CWebApplication->processRequest()
#2 D:\zyweb\pcb_logistics\website\home\index.php(38): CApplication->run()
#3 {main}
REQUEST_URI=/1
HTTP_REFERER=http://www.wllogistics.com/orders/list
---
2017/11/22 14:02:07 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "1".' in D:\zyweb\Yii1\web\CWebApplication.php:286
Stack trace:
#0 D:\zyweb\Yii1\web\CWebApplication.php(141): CWebApplication->runController('1')
#1 D:\zyweb\Yii1\base\CApplication.php(180): CWebApplication->processRequest()
#2 D:\zyweb\pcb_logistics\website\home\index.php(38): CApplication->run()
#3 {main}
REQUEST_URI=/1
HTTP_REFERER=http://www.wllogistics.com/orders/list
---
2017/11/22 14:02:07 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "1".' in D:\zyweb\Yii1\web\CWebApplication.php:286
Stack trace:
#0 D:\zyweb\Yii1\web\CWebApplication.php(141): CWebApplication->runController('1')
#1 D:\zyweb\Yii1\base\CApplication.php(180): CWebApplication->processRequest()
#2 D:\zyweb\pcb_logistics\website\home\index.php(38): CApplication->run()
#3 {main}
REQUEST_URI=/1
HTTP_REFERER=http://www.wllogistics.com/orders/list
---
2017/11/22 14:02:08 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "1".' in D:\zyweb\Yii1\web\CWebApplication.php:286
Stack trace:
#0 D:\zyweb\Yii1\web\CWebApplication.php(141): CWebApplication->runController('1')
#1 D:\zyweb\Yii1\base\CApplication.php(180): CWebApplication->processRequest()
#2 D:\zyweb\pcb_logistics\website\home\index.php(38): CApplication->run()
#3 {main}
REQUEST_URI=/1
HTTP_REFERER=http://www.wllogistics.com/orders/list
---
2017/11/22 14:02:08 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "favicon.ico".' in D:\zyweb\Yii1\web\CWebApplication.php:286
Stack trace:
#0 D:\zyweb\Yii1\web\CWebApplication.php(141): CWebApplication->runController('favicon.ico')
#1 D:\zyweb\Yii1\base\CApplication.php(180): CWebApplication->processRequest()
#2 D:\zyweb\pcb_logistics\website\home\index.php(38): CApplication->run()
#3 {main}
REQUEST_URI=/favicon.ico
HTTP_REFERER=http://www.wllogistics.com/orders/list
---
2017/11/22 14:02:08 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "1".' in D:\zyweb\Yii1\web\CWebApplication.php:286
Stack trace:
#0 D:\zyweb\Yii1\web\CWebApplication.php(141): CWebApplication->runController('1')
#1 D:\zyweb\Yii1\base\CApplication.php(180): CWebApplication->processRequest()
#2 D:\zyweb\pcb_logistics\website\home\index.php(38): CApplication->run()
#3 {main}
REQUEST_URI=/1
HTTP_REFERER=http://www.wllogistics.com/orders/list
---
2017/11/22 14:02:09 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "favicon.ico".' in D:\zyweb\Yii1\web\CWebApplication.php:286
Stack trace:
#0 D:\zyweb\Yii1\web\CWebApplication.php(141): CWebApplication->runController('favicon.ico')
#1 D:\zyweb\Yii1\base\CApplication.php(180): CWebApplication->processRequest()
#2 D:\zyweb\pcb_logistics\website\home\index.php(38): CApplication->run()
#3 {main}
REQUEST_URI=/favicon.ico
HTTP_REFERER=http://www.wllogistics.com/orders/list
---
2017/11/22 14:02:09 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "1".' in D:\zyweb\Yii1\web\CWebApplication.php:286
Stack trace:
#0 D:\zyweb\Yii1\web\CWebApplication.php(141): CWebApplication->runController('1')
#1 D:\zyweb\Yii1\base\CApplication.php(180): CWebApplication->processRequest()
#2 D:\zyweb\pcb_logistics\website\home\index.php(38): CApplication->run()
#3 {main}
REQUEST_URI=/1
HTTP_REFERER=http://www.wllogistics.com/orders/list
---
2017/11/22 14:08:52 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "1".' in D:\zyweb\Yii1\web\CWebApplication.php:286
Stack trace:
#0 D:\zyweb\Yii1\web\CWebApplication.php(141): CWebApplication->runController('1')
#1 D:\zyweb\Yii1\base\CApplication.php(180): CWebApplication->processRequest()
#2 D:\zyweb\pcb_logistics\website\home\index.php(38): CApplication->run()
#3 {main}
REQUEST_URI=/1
HTTP_REFERER=http://www.wllogistics.com/orders/list
---
2017/11/22 14:10:26 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "1".' in D:\zyweb\Yii1\web\CWebApplication.php:286
Stack trace:
#0 D:\zyweb\Yii1\web\CWebApplication.php(141): CWebApplication->runController('1')
#1 D:\zyweb\Yii1\base\CApplication.php(180): CWebApplication->processRequest()
#2 D:\zyweb\pcb_logistics\website\home\index.php(38): CApplication->run()
#3 {main}
REQUEST_URI=/1
HTTP_REFERER=http://www.wllogistics.com/orders/list
---
2017/11/22 14:10:27 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "1".' in D:\zyweb\Yii1\web\CWebApplication.php:286
Stack trace:
#0 D:\zyweb\Yii1\web\CWebApplication.php(141): CWebApplication->runController('1')
#1 D:\zyweb\Yii1\base\CApplication.php(180): CWebApplication->processRequest()
#2 D:\zyweb\pcb_logistics\website\home\index.php(38): CApplication->run()
#3 {main}
REQUEST_URI=/1
HTTP_REFERER=http://www.wllogistics.com/orders/list
---
2017/11/22 14:10:27 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "1".' in D:\zyweb\Yii1\web\CWebApplication.php:286
Stack trace:
#0 D:\zyweb\Yii1\web\CWebApplication.php(141): CWebApplication->runController('1')
#1 D:\zyweb\Yii1\base\CApplication.php(180): CWebApplication->processRequest()
#2 D:\zyweb\pcb_logistics\website\home\index.php(38): CApplication->run()
#3 {main}
REQUEST_URI=/1
HTTP_REFERER=http://www.wllogistics.com/orders/list
---
2017/11/22 14:10:27 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "1".' in D:\zyweb\Yii1\web\CWebApplication.php:286
Stack trace:
#0 D:\zyweb\Yii1\web\CWebApplication.php(141): CWebApplication->runController('1')
#1 D:\zyweb\Yii1\base\CApplication.php(180): CWebApplication->processRequest()
#2 D:\zyweb\pcb_logistics\website\home\index.php(38): CApplication->run()
#3 {main}
REQUEST_URI=/1
HTTP_REFERER=http://www.wllogistics.com/orders/list
---
2017/11/22 14:10:27 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "1".' in D:\zyweb\Yii1\web\CWebApplication.php:286
Stack trace:
#0 D:\zyweb\Yii1\web\CWebApplication.php(141): CWebApplication->runController('1')
#1 D:\zyweb\Yii1\base\CApplication.php(180): CWebApplication->processRequest()
#2 D:\zyweb\pcb_logistics\website\home\index.php(38): CApplication->run()
#3 {main}
REQUEST_URI=/1
HTTP_REFERER=http://www.wllogistics.com/orders/list
---
2017/11/22 14:10:28 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "1".' in D:\zyweb\Yii1\web\CWebApplication.php:286
Stack trace:
#0 D:\zyweb\Yii1\web\CWebApplication.php(141): CWebApplication->runController('1')
#1 D:\zyweb\Yii1\base\CApplication.php(180): CWebApplication->processRequest()
#2 D:\zyweb\pcb_logistics\website\home\index.php(38): CApplication->run()
#3 {main}
REQUEST_URI=/1
HTTP_REFERER=http://www.wllogistics.com/orders/list
---
2017/11/22 14:10:28 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "1".' in D:\zyweb\Yii1\web\CWebApplication.php:286
Stack trace:
#0 D:\zyweb\Yii1\web\CWebApplication.php(141): CWebApplication->runController('1')
#1 D:\zyweb\Yii1\base\CApplication.php(180): CWebApplication->processRequest()
#2 D:\zyweb\pcb_logistics\website\home\index.php(38): CApplication->run()
#3 {main}
REQUEST_URI=/1
HTTP_REFERER=http://www.wllogistics.com/orders/list
---
2017/11/22 14:10:28 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "1".' in D:\zyweb\Yii1\web\CWebApplication.php:286
Stack trace:
#0 D:\zyweb\Yii1\web\CWebApplication.php(141): CWebApplication->runController('1')
#1 D:\zyweb\Yii1\base\CApplication.php(180): CWebApplication->processRequest()
#2 D:\zyweb\pcb_logistics\website\home\index.php(38): CApplication->run()
#3 {main}
REQUEST_URI=/1
HTTP_REFERER=http://www.wllogistics.com/orders/list
---
2017/11/22 14:10:28 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "1".' in D:\zyweb\Yii1\web\CWebApplication.php:286
Stack trace:
#0 D:\zyweb\Yii1\web\CWebApplication.php(141): CWebApplication->runController('1')
#1 D:\zyweb\Yii1\base\CApplication.php(180): CWebApplication->processRequest()
#2 D:\zyweb\pcb_logistics\website\home\index.php(38): CApplication->run()
#3 {main}
REQUEST_URI=/1
HTTP_REFERER=http://www.wllogistics.com/orders/list
---
2017/11/22 14:10:28 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "1".' in D:\zyweb\Yii1\web\CWebApplication.php:286
Stack trace:
#0 D:\zyweb\Yii1\web\CWebApplication.php(141): CWebApplication->runController('1')
#1 D:\zyweb\Yii1\base\CApplication.php(180): CWebApplication->processRequest()
#2 D:\zyweb\pcb_logistics\website\home\index.php(38): CApplication->run()
#3 {main}
REQUEST_URI=/1
HTTP_REFERER=http://www.wllogistics.com/orders/list
---
2017/11/22 14:10:29 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "1".' in D:\zyweb\Yii1\web\CWebApplication.php:286
Stack trace:
#0 D:\zyweb\Yii1\web\CWebApplication.php(141): CWebApplication->runController('1')
#1 D:\zyweb\Yii1\base\CApplication.php(180): CWebApplication->processRequest()
#2 D:\zyweb\pcb_logistics\website\home\index.php(38): CApplication->run()
#3 {main}
REQUEST_URI=/1
HTTP_REFERER=http://www.wllogistics.com/orders/list
---
2017/11/22 14:10:29 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "1".' in D:\zyweb\Yii1\web\CWebApplication.php:286
Stack trace:
#0 D:\zyweb\Yii1\web\CWebApplication.php(141): CWebApplication->runController('1')
#1 D:\zyweb\Yii1\base\CApplication.php(180): CWebApplication->processRequest()
#2 D:\zyweb\pcb_logistics\website\home\index.php(38): CApplication->run()
#3 {main}
REQUEST_URI=/1
HTTP_REFERER=http://www.wllogistics.com/orders/list
---
2017/11/22 14:10:29 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "1".' in D:\zyweb\Yii1\web\CWebApplication.php:286
Stack trace:
#0 D:\zyweb\Yii1\web\CWebApplication.php(141): CWebApplication->runController('1')
#1 D:\zyweb\Yii1\base\CApplication.php(180): CWebApplication->processRequest()
#2 D:\zyweb\pcb_logistics\website\home\index.php(38): CApplication->run()
#3 {main}
REQUEST_URI=/1
HTTP_REFERER=http://www.wllogistics.com/orders/list
---
2017/11/22 14:10:29 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "1".' in D:\zyweb\Yii1\web\CWebApplication.php:286
Stack trace:
#0 D:\zyweb\Yii1\web\CWebApplication.php(141): CWebApplication->runController('1')
#1 D:\zyweb\Yii1\base\CApplication.php(180): CWebApplication->processRequest()
#2 D:\zyweb\pcb_logistics\website\home\index.php(38): CApplication->run()
#3 {main}
REQUEST_URI=/1
HTTP_REFERER=http://www.wllogistics.com/orders/list
---
2017/11/22 14:10:29 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "1".' in D:\zyweb\Yii1\web\CWebApplication.php:286
Stack trace:
#0 D:\zyweb\Yii1\web\CWebApplication.php(141): CWebApplication->runController('1')
#1 D:\zyweb\Yii1\base\CApplication.php(180): CWebApplication->processRequest()
#2 D:\zyweb\pcb_logistics\website\home\index.php(38): CApplication->run()
#3 {main}
REQUEST_URI=/1
HTTP_REFERER=http://www.wllogistics.com/orders/list
---
2017/11/22 14:10:30 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "1".' in D:\zyweb\Yii1\web\CWebApplication.php:286
Stack trace:
#0 D:\zyweb\Yii1\web\CWebApplication.php(141): CWebApplication->runController('1')
#1 D:\zyweb\Yii1\base\CApplication.php(180): CWebApplication->processRequest()
#2 D:\zyweb\pcb_logistics\website\home\index.php(38): CApplication->run()
#3 {main}
REQUEST_URI=/1
HTTP_REFERER=http://www.wllogistics.com/orders/list
---
2017/11/22 14:10:30 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "1".' in D:\zyweb\Yii1\web\CWebApplication.php:286
Stack trace:
#0 D:\zyweb\Yii1\web\CWebApplication.php(141): CWebApplication->runController('1')
#1 D:\zyweb\Yii1\base\CApplication.php(180): CWebApplication->processRequest()
#2 D:\zyweb\pcb_logistics\website\home\index.php(38): CApplication->run()
#3 {main}
REQUEST_URI=/1
HTTP_REFERER=http://www.wllogistics.com/orders/list
---
2017/11/22 14:10:30 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "1".' in D:\zyweb\Yii1\web\CWebApplication.php:286
Stack trace:
#0 D:\zyweb\Yii1\web\CWebApplication.php(141): CWebApplication->runController('1')
#1 D:\zyweb\Yii1\base\CApplication.php(180): CWebApplication->processRequest()
#2 D:\zyweb\pcb_logistics\website\home\index.php(38): CApplication->run()
#3 {main}
REQUEST_URI=/1
HTTP_REFERER=http://www.wllogistics.com/orders/list
---
2017/11/22 14:10:30 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "1".' in D:\zyweb\Yii1\web\CWebApplication.php:286
Stack trace:
#0 D:\zyweb\Yii1\web\CWebApplication.php(141): CWebApplication->runController('1')
#1 D:\zyweb\Yii1\base\CApplication.php(180): CWebApplication->processRequest()
#2 D:\zyweb\pcb_logistics\website\home\index.php(38): CApplication->run()
#3 {main}
REQUEST_URI=/1
HTTP_REFERER=http://www.wllogistics.com/orders/list
---
2017/11/22 14:10:30 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "1".' in D:\zyweb\Yii1\web\CWebApplication.php:286
Stack trace:
#0 D:\zyweb\Yii1\web\CWebApplication.php(141): CWebApplication->runController('1')
#1 D:\zyweb\Yii1\base\CApplication.php(180): CWebApplication->processRequest()
#2 D:\zyweb\pcb_logistics\website\home\index.php(38): CApplication->run()
#3 {main}
REQUEST_URI=/1
HTTP_REFERER=http://www.wllogistics.com/orders/list
---
2017/11/22 14:11:49 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "1".' in D:\zyweb\Yii1\web\CWebApplication.php:286
Stack trace:
#0 D:\zyweb\Yii1\web\CWebApplication.php(141): CWebApplication->runController('1')
#1 D:\zyweb\Yii1\base\CApplication.php(180): CWebApplication->processRequest()
#2 D:\zyweb\pcb_logistics\website\home\index.php(38): CApplication->run()
#3 {main}
REQUEST_URI=/1
HTTP_REFERER=http://www.wllogistics.com/orders/list
---
2017/11/22 14:28:00 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "1".' in D:\zyweb\Yii1\web\CWebApplication.php:286
Stack trace:
#0 D:\zyweb\Yii1\web\CWebApplication.php(141): CWebApplication->runController('1')
#1 D:\zyweb\Yii1\base\CApplication.php(180): CWebApplication->processRequest()
#2 D:\zyweb\pcb_logistics\website\home\index.php(38): CApplication->run()
#3 {main}
REQUEST_URI=/1
HTTP_REFERER=http://www.wllogistics.com/router/routerlist
---
2017/11/22 14:28:02 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "1".' in D:\zyweb\Yii1\web\CWebApplication.php:286
Stack trace:
#0 D:\zyweb\Yii1\web\CWebApplication.php(141): CWebApplication->runController('1')
#1 D:\zyweb\Yii1\base\CApplication.php(180): CWebApplication->processRequest()
#2 D:\zyweb\pcb_logistics\website\home\index.php(38): CApplication->run()
#3 {main}
REQUEST_URI=/1
HTTP_REFERER=http://www.wllogistics.com/goods/list
---
2017/11/22 14:28:03 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "1".' in D:\zyweb\Yii1\web\CWebApplication.php:286
Stack trace:
#0 D:\zyweb\Yii1\web\CWebApplication.php(141): CWebApplication->runController('1')
#1 D:\zyweb\Yii1\base\CApplication.php(180): CWebApplication->processRequest()
#2 D:\zyweb\pcb_logistics\website\home\index.php(38): CApplication->run()
#3 {main}
REQUEST_URI=/1
HTTP_REFERER=http://www.wllogistics.com/address/list
---
2017/11/22 14:28:21 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "1".' in D:\zyweb\Yii1\web\CWebApplication.php:286
Stack trace:
#0 D:\zyweb\Yii1\web\CWebApplication.php(141): CWebApplication->runController('1')
#1 D:\zyweb\Yii1\base\CApplication.php(180): CWebApplication->processRequest()
#2 D:\zyweb\pcb_logistics\website\home\index.php(38): CApplication->run()
#3 {main}
REQUEST_URI=/1
HTTP_REFERER=http://www.wllogistics.com/orders/list
---
2017/11/22 14:28:23 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "1".' in D:\zyweb\Yii1\web\CWebApplication.php:286
Stack trace:
#0 D:\zyweb\Yii1\web\CWebApplication.php(141): CWebApplication->runController('1')
#1 D:\zyweb\Yii1\base\CApplication.php(180): CWebApplication->processRequest()
#2 D:\zyweb\pcb_logistics\website\home\index.php(38): CApplication->run()
#3 {main}
REQUEST_URI=/1
HTTP_REFERER=http://www.wllogistics.com/orders/list
---
2017/11/22 14:30:25 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "1".' in D:\zyweb\Yii1\web\CWebApplication.php:286
Stack trace:
#0 D:\zyweb\Yii1\web\CWebApplication.php(141): CWebApplication->runController('1')
#1 D:\zyweb\Yii1\base\CApplication.php(180): CWebApplication->processRequest()
#2 D:\zyweb\pcb_logistics\website\home\index.php(38): CApplication->run()
#3 {main}
REQUEST_URI=/1
HTTP_REFERER=http://www.wllogistics.com/orders/list
---
2017/11/22 14:34:29 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "1".' in D:\zyweb\Yii1\web\CWebApplication.php:286
Stack trace:
#0 D:\zyweb\Yii1\web\CWebApplication.php(141): CWebApplication->runController('1')
#1 D:\zyweb\Yii1\base\CApplication.php(180): CWebApplication->processRequest()
#2 D:\zyweb\pcb_logistics\website\home\index.php(38): CApplication->run()
#3 {main}
REQUEST_URI=/1
HTTP_REFERER=http://www.wllogistics.com/orders/list
---
2017/11/22 14:36:41 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "1".' in D:\zyweb\Yii1\web\CWebApplication.php:286
Stack trace:
#0 D:\zyweb\Yii1\web\CWebApplication.php(141): CWebApplication->runController('1')
#1 D:\zyweb\Yii1\base\CApplication.php(180): CWebApplication->processRequest()
#2 D:\zyweb\pcb_logistics\website\home\index.php(38): CApplication->run()
#3 {main}
REQUEST_URI=/1
HTTP_REFERER=http://www.wllogistics.com/orders/list
---
2017/11/22 14:36:59 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "1".' in D:\zyweb\Yii1\web\CWebApplication.php:286
Stack trace:
#0 D:\zyweb\Yii1\web\CWebApplication.php(141): CWebApplication->runController('1')
#1 D:\zyweb\Yii1\base\CApplication.php(180): CWebApplication->processRequest()
#2 D:\zyweb\pcb_logistics\website\home\index.php(38): CApplication->run()
#3 {main}
REQUEST_URI=/1
HTTP_REFERER=http://www.wllogistics.com/orders/list
---
2017/11/22 14:37:07 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "1".' in D:\zyweb\Yii1\web\CWebApplication.php:286
Stack trace:
#0 D:\zyweb\Yii1\web\CWebApplication.php(141): CWebApplication->runController('1')
#1 D:\zyweb\Yii1\base\CApplication.php(180): CWebApplication->processRequest()
#2 D:\zyweb\pcb_logistics\website\home\index.php(38): CApplication->run()
#3 {main}
REQUEST_URI=/1
HTTP_REFERER=http://www.wllogistics.com/orders/list
---
2017/11/22 14:37:25 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "1".' in D:\zyweb\Yii1\web\CWebApplication.php:286
Stack trace:
#0 D:\zyweb\Yii1\web\CWebApplication.php(141): CWebApplication->runController('1')
#1 D:\zyweb\Yii1\base\CApplication.php(180): CWebApplication->processRequest()
#2 D:\zyweb\pcb_logistics\website\home\index.php(38): CApplication->run()
#3 {main}
REQUEST_URI=/1
HTTP_REFERER=http://www.wllogistics.com/orders/list
---
2017/11/22 14:37:39 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "1".' in D:\zyweb\Yii1\web\CWebApplication.php:286
Stack trace:
#0 D:\zyweb\Yii1\web\CWebApplication.php(141): CWebApplication->runController('1')
#1 D:\zyweb\Yii1\base\CApplication.php(180): CWebApplication->processRequest()
#2 D:\zyweb\pcb_logistics\website\home\index.php(38): CApplication->run()
#3 {main}
REQUEST_URI=/1
HTTP_REFERER=http://www.wllogistics.com/orders/list
---
2017/11/22 15:03:39 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "1".' in D:\zyweb\Yii1\web\CWebApplication.php:286
Stack trace:
#0 D:\zyweb\Yii1\web\CWebApplication.php(141): CWebApplication->runController('1')
#1 D:\zyweb\Yii1\base\CApplication.php(180): CWebApplication->processRequest()
#2 D:\zyweb\pcb_logistics\website\home\index.php(38): CApplication->run()
#3 {main}
REQUEST_URI=/1
HTTP_REFERER=http://www.wllogistics.com/orders/list
---
2017/11/22 15:03:40 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "1".' in D:\zyweb\Yii1\web\CWebApplication.php:286
Stack trace:
#0 D:\zyweb\Yii1\web\CWebApplication.php(141): CWebApplication->runController('1')
#1 D:\zyweb\Yii1\base\CApplication.php(180): CWebApplication->processRequest()
#2 D:\zyweb\pcb_logistics\website\home\index.php(38): CApplication->run()
#3 {main}
REQUEST_URI=/1
HTTP_REFERER=http://www.wllogistics.com/orders/list
---
2017/11/22 15:03:40 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "1".' in D:\zyweb\Yii1\web\CWebApplication.php:286
Stack trace:
#0 D:\zyweb\Yii1\web\CWebApplication.php(141): CWebApplication->runController('1')
#1 D:\zyweb\Yii1\base\CApplication.php(180): CWebApplication->processRequest()
#2 D:\zyweb\pcb_logistics\website\home\index.php(38): CApplication->run()
#3 {main}
REQUEST_URI=/1
HTTP_REFERER=http://www.wllogistics.com/orders/list
---
2017/11/22 15:03:40 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "1".' in D:\zyweb\Yii1\web\CWebApplication.php:286
Stack trace:
#0 D:\zyweb\Yii1\web\CWebApplication.php(141): CWebApplication->runController('1')
#1 D:\zyweb\Yii1\base\CApplication.php(180): CWebApplication->processRequest()
#2 D:\zyweb\pcb_logistics\website\home\index.php(38): CApplication->run()
#3 {main}
REQUEST_URI=/1
HTTP_REFERER=http://www.wllogistics.com/orders/list
---
2017/11/22 15:03:40 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "1".' in D:\zyweb\Yii1\web\CWebApplication.php:286
Stack trace:
#0 D:\zyweb\Yii1\web\CWebApplication.php(141): CWebApplication->runController('1')
#1 D:\zyweb\Yii1\base\CApplication.php(180): CWebApplication->processRequest()
#2 D:\zyweb\pcb_logistics\website\home\index.php(38): CApplication->run()
#3 {main}
REQUEST_URI=/1
HTTP_REFERER=http://www.wllogistics.com/orders/list
---
2017/11/22 15:03:41 [error] [exception.CException] exception 'CException' with message 'Orders and its behaviors do not have a method or closure named "getByGoodsId".' in D:\zyweb\Yii1\base\CComponent.php:266
Stack trace:
#0 D:\zyweb\pcb_logistics\applications\home\controllers\OrdersController.php(49): CComponent->__call('getByGoodsId', Array)
#1 D:\zyweb\pcb_logistics\applications\home\controllers\OrdersController.php(49): Orders->getByGoodsId('14')
#2 D:\zyweb\Yii1\web\actions\CInlineAction.php(49): OrdersController->actionEdit()
#3 D:\zyweb\Yii1\web\CController.php(308): CInlineAction->runWithParams(Array)
#4 D:\zyweb\Yii1\web\CController.php(286): CController->runAction(Object(CInlineAction))
#5 D:\zyweb\Yii1\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#6 D:\zyweb\pcb_logistics\applications\home\components\Controller.php(30): CController->run('edit')
#7 D:\zyweb\Yii1\web\CWebApplication.php(282): Controller->run('edit')
#8 D:\zyweb\Yii1\web\CWebApplication.php(141): CWebApplication->runController('orders/edit/id/...')
#9 D:\zyweb\Yii1\base\CApplication.php(180): CWebApplication->processRequest()
#10 D:\zyweb\pcb_logistics\website\home\index.php(38): CApplication->run()
#11 {main}
REQUEST_URI=/orders/edit/id/14/again/1
HTTP_REFERER=http://www.wllogistics.com/orders/list
---
2017/11/22 15:03:57 [error] [php] include(OrdersGoods.php): failed to open stream: No such file or directory (D:\zyweb\Yii1\YiiBase.php:427)
Stack trace:
#0 D:\zyweb\pcb_logistics\applications\home\controllers\OrdersController.php(49): spl_autoload_call()
#1 D:\zyweb\Yii1\web\actions\CInlineAction.php(49): OrdersController->actionEdit()
#2 D:\zyweb\Yii1\web\CController.php(308): CInlineAction->runWithParams()
#3 D:\zyweb\Yii1\web\CController.php(286): OrdersController->runAction()
#4 D:\zyweb\Yii1\web\CController.php(265): OrdersController->runActionWithFilters()
#5 D:\zyweb\pcb_logistics\applications\home\components\Controller.php(30): OrdersController->run()
#6 D:\zyweb\Yii1\web\CWebApplication.php(282): OrdersController->run()
#7 D:\zyweb\Yii1\web\CWebApplication.php(141): CWebApplication->runController()
#8 D:\zyweb\Yii1\base\CApplication.php(180): CWebApplication->processRequest()
#9 D:\zyweb\pcb_logistics\website\home\index.php(38): CWebApplication->run()
REQUEST_URI=/orders/edit/id/14/again/1
in D:\zyweb\pcb_logistics\applications\home\controllers\OrdersController.php (49)
in D:\zyweb\pcb_logistics\applications\home\components\Controller.php (30)
in D:\zyweb\pcb_logistics\website\home\index.php (38)
2017/11/22 15:06:29 [error] [php] Illegal string offset 'goodsName' (D:\zyweb\pcb_logistics\applications\home\views\orders\freGoods.php:7)
Stack trace:
#0 D:\zyweb\Yii1\web\CController.php(869): OrdersController->renderFile()
#1 D:\zyweb\pcb_logistics\applications\home\views\orders\edit.php(147): OrdersController->renderPartial()
#2 D:\zyweb\Yii1\web\CBaseController.php(126): require()
#3 D:\zyweb\Yii1\web\CBaseController.php(95): OrdersController->renderInternal()
#4 D:\zyweb\Yii1\web\CController.php(869): OrdersController->renderFile()
#5 D:\zyweb\Yii1\web\CController.php(782): OrdersController->renderPartial()
#6 D:\zyweb\pcb_logistics\applications\home\components\Controller.php(52): OrdersController->render()
#7 D:\zyweb\pcb_logistics\applications\home\controllers\OrdersController.php(78): OrdersController->render()
#8 D:\zyweb\Yii1\web\actions\CInlineAction.php(49): OrdersController->actionEdit()
#9 D:\zyweb\Yii1\web\CController.php(308): CInlineAction->runWithParams()
#10 D:\zyweb\Yii1\web\CController.php(286): OrdersController->runAction()
#11 D:\zyweb\Yii1\web\CController.php(265): OrdersController->runActionWithFilters()
#12 D:\zyweb\pcb_logistics\applications\home\components\Controller.php(30): OrdersController->run()
#13 D:\zyweb\Yii1\web\CWebApplication.php(282): OrdersController->run()
#14 D:\zyweb\Yii1\web\CWebApplication.php(141): CWebApplication->runController()
#15 D:\zyweb\Yii1\base\CApplication.php(180): CWebApplication->processRequest()
#16 D:\zyweb\pcb_logistics\website\home\index.php(38): CWebApplication->run()
REQUEST_URI=/orders/edit/id/14/again/1
in D:\zyweb\pcb_logistics\applications\home\views\orders\freGoods.php (7)
in D:\zyweb\pcb_logistics\applications\home\views\orders\edit.php (147)
in D:\zyweb\pcb_logistics\applications\home\components\Controller.php (52)
2017/11/22 15:14:44 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "1".' in D:\zyweb\Yii1\web\CWebApplication.php:286
Stack trace:
#0 D:\zyweb\Yii1\web\CWebApplication.php(141): CWebApplication->runController('1')
#1 D:\zyweb\Yii1\base\CApplication.php(180): CWebApplication->processRequest()
#2 D:\zyweb\pcb_logistics\website\home\index.php(38): CApplication->run()
#3 {main}
REQUEST_URI=/1
HTTP_REFERER=http://www.wllogistics.com/orders/list
---
2017/11/22 15:18:09 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "1".' in D:\zyweb\Yii1\web\CWebApplication.php:286
Stack trace:
#0 D:\zyweb\Yii1\web\CWebApplication.php(141): CWebApplication->runController('1')
#1 D:\zyweb\Yii1\base\CApplication.php(180): CWebApplication->processRequest()
#2 D:\zyweb\pcb_logistics\website\home\index.php(38): CApplication->run()
#3 {main}
REQUEST_URI=/1
HTTP_REFERER=http://www.wllogistics.com/Orders/list
---
2017/11/22 15:18:24 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "1".' in D:\zyweb\Yii1\web\CWebApplication.php:286
Stack trace:
#0 D:\zyweb\Yii1\web\CWebApplication.php(141): CWebApplication->runController('1')
#1 D:\zyweb\Yii1\base\CApplication.php(180): CWebApplication->processRequest()
#2 D:\zyweb\pcb_logistics\website\home\index.php(38): CApplication->run()
#3 {main}
REQUEST_URI=/1
HTTP_REFERER=http://www.wllogistics.com/orders/list
---
2017/11/22 15:20:45 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "1".' in D:\zyweb\Yii1\web\CWebApplication.php:286
Stack trace:
#0 D:\zyweb\Yii1\web\CWebApplication.php(141): CWebApplication->runController('1')
#1 D:\zyweb\Yii1\base\CApplication.php(180): CWebApplication->processRequest()
#2 D:\zyweb\pcb_logistics\website\home\index.php(38): CApplication->run()
#3 {main}
REQUEST_URI=/1
HTTP_REFERER=http://www.wllogistics.com/orders/list
---
2017/11/22 15:26:08 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "favicon.ico".' in D:\zyweb\Yii1\web\CWebApplication.php:286
Stack trace:
#0 D:\zyweb\Yii1\web\CWebApplication.php(141): CWebApplication->runController('favicon.ico')
#1 D:\zyweb\Yii1\base\CApplication.php(180): CWebApplication->processRequest()
#2 D:\zyweb\pcb_logistics\website\home\index.php(38): CApplication->run()
#3 {main}
REQUEST_URI=/favicon.ico
HTTP_REFERER=http://www.wllogistics.com/orders/edit
---
2017/11/22 15:26:08 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "favicon.ico".' in D:\zyweb\Yii1\web\CWebApplication.php:286
Stack trace:
#0 D:\zyweb\Yii1\web\CWebApplication.php(141): CWebApplication->runController('favicon.ico')
#1 D:\zyweb\Yii1\base\CApplication.php(180): CWebApplication->processRequest()
#2 D:\zyweb\pcb_logistics\website\home\index.php(38): CApplication->run()
#3 {main}
REQUEST_URI=/favicon.ico
HTTP_REFERER=http://www.wllogistics.com/orders/edit
---
2017/11/22 15:26:08 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "favicon.ico".' in D:\zyweb\Yii1\web\CWebApplication.php:286
Stack trace:
#0 D:\zyweb\Yii1\web\CWebApplication.php(141): CWebApplication->runController('favicon.ico')
#1 D:\zyweb\Yii1\base\CApplication.php(180): CWebApplication->processRequest()
#2 D:\zyweb\pcb_logistics\website\home\index.php(38): CApplication->run()
#3 {main}
REQUEST_URI=/favicon.ico
HTTP_REFERER=http://www.wllogistics.com/orders/edit
---
2017/11/22 15:26:08 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "favicon.ico".' in D:\zyweb\Yii1\web\CWebApplication.php:286
Stack trace:
#0 D:\zyweb\Yii1\web\CWebApplication.php(141): CWebApplication->runController('favicon.ico')
#1 D:\zyweb\Yii1\base\CApplication.php(180): CWebApplication->processRequest()
#2 D:\zyweb\pcb_logistics\website\home\index.php(38): CApplication->run()
#3 {main}
REQUEST_URI=/favicon.ico
HTTP_REFERER=http://www.wllogistics.com/orders/edit
---
2017/11/22 15:26:09 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "favicon.ico".' in D:\zyweb\Yii1\web\CWebApplication.php:286
Stack trace:
#0 D:\zyweb\Yii1\web\CWebApplication.php(141): CWebApplication->runController('favicon.ico')
#1 D:\zyweb\Yii1\base\CApplication.php(180): CWebApplication->processRequest()
#2 D:\zyweb\pcb_logistics\website\home\index.php(38): CApplication->run()
#3 {main}
REQUEST_URI=/favicon.ico
HTTP_REFERER=http://www.wllogistics.com/orders/edit
---
2017/11/22 15:26:39 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "1".' in D:\zyweb\Yii1\web\CWebApplication.php:286
Stack trace:
#0 D:\zyweb\Yii1\web\CWebApplication.php(141): CWebApplication->runController('1')
#1 D:\zyweb\Yii1\base\CApplication.php(180): CWebApplication->processRequest()
#2 D:\zyweb\pcb_logistics\website\home\index.php(38): CApplication->run()
#3 {main}
REQUEST_URI=/1
HTTP_REFERER=http://www.wllogistics.com/address/list
---
2017/11/22 15:26:41 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "1".' in D:\zyweb\Yii1\web\CWebApplication.php:286
Stack trace:
#0 D:\zyweb\Yii1\web\CWebApplication.php(141): CWebApplication->runController('1')
#1 D:\zyweb\Yii1\base\CApplication.php(180): CWebApplication->processRequest()
#2 D:\zyweb\pcb_logistics\website\home\index.php(38): CApplication->run()
#3 {main}
REQUEST_URI=/1
HTTP_REFERER=http://www.wllogistics.com/orders/list
---
2017/11/22 15:26:48 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "1".' in D:\zyweb\Yii1\web\CWebApplication.php:286
Stack trace:
#0 D:\zyweb\Yii1\web\CWebApplication.php(141): CWebApplication->runController('1')
#1 D:\zyweb\Yii1\base\CApplication.php(180): CWebApplication->processRequest()
#2 D:\zyweb\pcb_logistics\website\home\index.php(38): CApplication->run()
#3 {main}
REQUEST_URI=/1
HTTP_REFERER=http://www.wllogistics.com/address/list
---
2017/11/22 15:26:53 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "1".' in D:\zyweb\Yii1\web\CWebApplication.php:286
Stack trace:
#0 D:\zyweb\Yii1\web\CWebApplication.php(141): CWebApplication->runController('1')
#1 D:\zyweb\Yii1\base\CApplication.php(180): CWebApplication->processRequest()
#2 D:\zyweb\pcb_logistics\website\home\index.php(38): CApplication->run()
#3 {main}
REQUEST_URI=/1
HTTP_REFERER=http://www.wllogistics.com/goods/list
---
2017/11/22 15:27:02 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "1".' in D:\zyweb\Yii1\web\CWebApplication.php:286
Stack trace:
#0 D:\zyweb\Yii1\web\CWebApplication.php(141): CWebApplication->runController('1')
#1 D:\zyweb\Yii1\base\CApplication.php(180): CWebApplication->processRequest()
#2 D:\zyweb\pcb_logistics\website\home\index.php(38): CApplication->run()
#3 {main}
REQUEST_URI=/1
HTTP_REFERER=http://www.wllogistics.com/router/routerlist
---
2017/11/22 15:27:04 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "1".' in D:\zyweb\Yii1\web\CWebApplication.php:286
Stack trace:
#0 D:\zyweb\Yii1\web\CWebApplication.php(141): CWebApplication->runController('1')
#1 D:\zyweb\Yii1\base\CApplication.php(180): CWebApplication->processRequest()
#2 D:\zyweb\pcb_logistics\website\home\index.php(38): CApplication->run()
#3 {main}
REQUEST_URI=/1
HTTP_REFERER=http://www.wllogistics.com/router/routerlist?RouterForm%5Bname%5D=
---
2017/11/22 15:28:54 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "favicon.ico".' in D:\zyweb\Yii1\web\CWebApplication.php:286
Stack trace:
#0 D:\zyweb\Yii1\web\CWebApplication.php(141): CWebApplication->runController('favicon.ico')
#1 D:\zyweb\Yii1\base\CApplication.php(180): CWebApplication->processRequest()
#2 D:\zyweb\pcb_logistics\website\home\index.php(38): CApplication->run()
#3 {main}
REQUEST_URI=/favicon.ico
HTTP_REFERER=http://www.wllogistics.com/router/editrouter
---
2017/11/22 15:28:54 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "favicon.ico".' in D:\zyweb\Yii1\web\CWebApplication.php:286
Stack trace:
#0 D:\zyweb\Yii1\web\CWebApplication.php(141): CWebApplication->runController('favicon.ico')
#1 D:\zyweb\Yii1\base\CApplication.php(180): CWebApplication->processRequest()
#2 D:\zyweb\pcb_logistics\website\home\index.php(38): CApplication->run()
#3 {main}
REQUEST_URI=/favicon.ico
HTTP_REFERER=http://www.wllogistics.com/router/editrouter
---
2017/11/22 15:38:11 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "1".' in D:\zyweb\Yii1\web\CWebApplication.php:286
Stack trace:
#0 D:\zyweb\Yii1\web\CWebApplication.php(141): CWebApplication->runController('1')
#1 D:\zyweb\Yii1\base\CApplication.php(180): CWebApplication->processRequest()
#2 D:\zyweb\pcb_logistics\website\home\index.php(38): CApplication->run()
#3 {main}
REQUEST_URI=/1
HTTP_REFERER=http://www.wllogistics.com/address/list
---
2017/11/22 16:28:30 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "1".' in D:\zyweb\Yii1\web\CWebApplication.php:286
Stack trace:
#0 D:\zyweb\Yii1\web\CWebApplication.php(141): CWebApplication->runController('1')
#1 D:\zyweb\Yii1\base\CApplication.php(180): CWebApplication->processRequest()
#2 D:\zyweb\pcb_logistics\website\home\index.php(38): CApplication->run()
#3 {main}
REQUEST_URI=/1
HTTP_REFERER=http://www.wllogistics.com/Orders/list
---
2017/11/22 16:39:57 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "1".' in D:\zyweb\Yii1\web\CWebApplication.php:286
Stack trace:
#0 D:\zyweb\Yii1\web\CWebApplication.php(141): CWebApplication->runController('1')
#1 D:\zyweb\Yii1\base\CApplication.php(180): CWebApplication->processRequest()
#2 D:\zyweb\pcb_logistics\website\home\index.php(38): CApplication->run()
#3 {main}
REQUEST_URI=/1
HTTP_REFERER=http://www.wllogistics.com/Orders/list
---
2017/11/22 16:39:58 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "1".' in D:\zyweb\Yii1\web\CWebApplication.php:286
Stack trace:
#0 D:\zyweb\Yii1\web\CWebApplication.php(141): CWebApplication->runController('1')
#1 D:\zyweb\Yii1\base\CApplication.php(180): CWebApplication->processRequest()
#2 D:\zyweb\pcb_logistics\website\home\index.php(38): CApplication->run()
#3 {main}
REQUEST_URI=/1
HTTP_REFERER=http://www.wllogistics.com/Orders/list
---
2017/11/22 16:39:58 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "1".' in D:\zyweb\Yii1\web\CWebApplication.php:286
Stack trace:
#0 D:\zyweb\Yii1\web\CWebApplication.php(141): CWebApplication->runController('1')
#1 D:\zyweb\Yii1\base\CApplication.php(180): CWebApplication->processRequest()
#2 D:\zyweb\pcb_logistics\website\home\index.php(38): CApplication->run()
#3 {main}
REQUEST_URI=/1
HTTP_REFERER=http://www.wllogistics.com/Orders/list
---
2017/11/22 16:40:00 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "1".' in D:\zyweb\Yii1\web\CWebApplication.php:286
Stack trace:
#0 D:\zyweb\Yii1\web\CWebApplication.php(141): CWebApplication->runController('1')
#1 D:\zyweb\Yii1\base\CApplication.php(180): CWebApplication->processRequest()
#2 D:\zyweb\pcb_logistics\website\home\index.php(38): CApplication->run()
#3 {main}
REQUEST_URI=/1
HTTP_REFERER=http://www.wllogistics.com/Orders/list
---
2017/11/22 16:48:06 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "1".' in D:\zyweb\Yii1\web\CWebApplication.php:286
Stack trace:
#0 D:\zyweb\Yii1\web\CWebApplication.php(141): CWebApplication->runController('1')
#1 D:\zyweb\Yii1\base\CApplication.php(180): CWebApplication->processRequest()
#2 D:\zyweb\pcb_logistics\website\home\index.php(38): CApplication->run()
#3 {main}
REQUEST_URI=/1
HTTP_REFERER=http://www.wllogistics.com/Orders/list
---
2017/11/22 16:48:41 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "1".' in D:\zyweb\Yii1\web\CWebApplication.php:286
Stack trace:
#0 D:\zyweb\Yii1\web\CWebApplication.php(141): CWebApplication->runController('1')
#1 D:\zyweb\Yii1\base\CApplication.php(180): CWebApplication->processRequest()
#2 D:\zyweb\pcb_logistics\website\home\index.php(38): CApplication->run()
#3 {main}
REQUEST_URI=/1
HTTP_REFERER=http://www.wllogistics.com/Orders/list
---
2017/11/22 17:04:58 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "1".' in D:\zyweb\Yii1\web\CWebApplication.php:286
Stack trace:
#0 D:\zyweb\Yii1\web\CWebApplication.php(141): CWebApplication->runController('1')
#1 D:\zyweb\Yii1\base\CApplication.php(180): CWebApplication->processRequest()
#2 D:\zyweb\pcb_logistics\website\home\index.php(38): CApplication->run()
#3 {main}
REQUEST_URI=/1
HTTP_REFERER=http://www.wllogistics.com/address/list
---
2017/11/22 17:08:56 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "1".' in D:\zyweb\Yii1\web\CWebApplication.php:286
Stack trace:
#0 D:\zyweb\Yii1\web\CWebApplication.php(141): CWebApplication->runController('1')
#1 D:\zyweb\Yii1\base\CApplication.php(180): CWebApplication->processRequest()
#2 D:\zyweb\pcb_logistics\website\home\index.php(38): CApplication->run()
#3 {main}
REQUEST_URI=/1
HTTP_REFERER=http://www.wllogistics.com/address/list
---
2017/11/22 17:08:57 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "1".' in D:\zyweb\Yii1\web\CWebApplication.php:286
Stack trace:
#0 D:\zyweb\Yii1\web\CWebApplication.php(141): CWebApplication->runController('1')
#1 D:\zyweb\Yii1\base\CApplication.php(180): CWebApplication->processRequest()
#2 D:\zyweb\pcb_logistics\website\home\index.php(38): CApplication->run()
#3 {main}
REQUEST_URI=/1
HTTP_REFERER=http://www.wllogistics.com/orders/list
---
2017/11/22 17:19:54 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "1".' in D:\zyweb\Yii1\web\CWebApplication.php:286
Stack trace:
#0 D:\zyweb\Yii1\web\CWebApplication.php(141): CWebApplication->runController('1')
#1 D:\zyweb\Yii1\base\CApplication.php(180): CWebApplication->processRequest()
#2 D:\zyweb\pcb_logistics\website\home\index.php(38): CApplication->run()
#3 {main}
REQUEST_URI=/1
HTTP_REFERER=http://www.wllogistics.com/orders/list
---
2017/11/22 17:22:17 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "1".' in D:\zyweb\Yii1\web\CWebApplication.php:286
Stack trace:
#0 D:\zyweb\Yii1\web\CWebApplication.php(141): CWebApplication->runController('1')
#1 D:\zyweb\Yii1\base\CApplication.php(180): CWebApplication->processRequest()
#2 D:\zyweb\pcb_logistics\website\home\index.php(38): CApplication->run()
#3 {main}
REQUEST_URI=/1
HTTP_REFERER=http://www.wllogistics.com/orders/list
---
2017/11/22 17:22:17 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "1".' in D:\zyweb\Yii1\web\CWebApplication.php:286
Stack trace:
#0 D:\zyweb\Yii1\web\CWebApplication.php(141): CWebApplication->runController('1')
#1 D:\zyweb\Yii1\base\CApplication.php(180): CWebApplication->processRequest()
#2 D:\zyweb\pcb_logistics\website\home\index.php(38): CApplication->run()
#3 {main}
REQUEST_URI=/1
HTTP_REFERER=http://www.wllogistics.com/orders/list
---
2017/11/22 17:22:17 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "1".' in D:\zyweb\Yii1\web\CWebApplication.php:286
Stack trace:
#0 D:\zyweb\Yii1\web\CWebApplication.php(141): CWebApplication->runController('1')
#1 D:\zyweb\Yii1\base\CApplication.php(180): CWebApplication->processRequest()
#2 D:\zyweb\pcb_logistics\website\home\index.php(38): CApplication->run()
#3 {main}
REQUEST_URI=/1
HTTP_REFERER=http://www.wllogistics.com/orders/list
---
2017/11/22 17:22:17 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "1".' in D:\zyweb\Yii1\web\CWebApplication.php:286
Stack trace:
#0 D:\zyweb\Yii1\web\CWebApplication.php(141): CWebApplication->runController('1')
#1 D:\zyweb\Yii1\base\CApplication.php(180): CWebApplication->processRequest()
#2 D:\zyweb\pcb_logistics\website\home\index.php(38): CApplication->run()
#3 {main}
REQUEST_URI=/1
HTTP_REFERER=http://www.wllogistics.com/orders/list
---
2017/11/22 17:24:00 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "1".' in D:\zyweb\Yii1\web\CWebApplication.php:286
Stack trace:
#0 D:\zyweb\Yii1\web\CWebApplication.php(141): CWebApplication->runController('1')
#1 D:\zyweb\Yii1\base\CApplication.php(180): CWebApplication->processRequest()
#2 D:\zyweb\pcb_logistics\website\home\index.php(38): CApplication->run()
#3 {main}
REQUEST_URI=/1
HTTP_REFERER=http://www.wllogistics.com/orders/list
---
2017/11/22 17:24:01 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "1".' in D:\zyweb\Yii1\web\CWebApplication.php:286
Stack trace:
#0 D:\zyweb\Yii1\web\CWebApplication.php(141): CWebApplication->runController('1')
#1 D:\zyweb\Yii1\base\CApplication.php(180): CWebApplication->processRequest()
#2 D:\zyweb\pcb_logistics\website\home\index.php(38): CApplication->run()
#3 {main}
REQUEST_URI=/1
HTTP_REFERER=http://www.wllogistics.com/goods/list
---
2017/11/22 17:24:02 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "1".' in D:\zyweb\Yii1\web\CWebApplication.php:286
Stack trace:
#0 D:\zyweb\Yii1\web\CWebApplication.php(141): CWebApplication->runController('1')
#1 D:\zyweb\Yii1\base\CApplication.php(180): CWebApplication->processRequest()
#2 D:\zyweb\pcb_logistics\website\home\index.php(38): CApplication->run()
#3 {main}
REQUEST_URI=/1
HTTP_REFERER=http://www.wllogistics.com/router/routerlist
---
2017/11/22 17:24:03 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "1".' in D:\zyweb\Yii1\web\CWebApplication.php:286
Stack trace:
#0 D:\zyweb\Yii1\web\CWebApplication.php(141): CWebApplication->runController('1')
#1 D:\zyweb\Yii1\base\CApplication.php(180): CWebApplication->processRequest()
#2 D:\zyweb\pcb_logistics\website\home\index.php(38): CApplication->run()
#3 {main}
REQUEST_URI=/1
HTTP_REFERER=http://www.wllogistics.com/goods/list
---
2017/11/22 17:24:04 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "1".' in D:\zyweb\Yii1\web\CWebApplication.php:286
Stack trace:
#0 D:\zyweb\Yii1\web\CWebApplication.php(141): CWebApplication->runController('1')
#1 D:\zyweb\Yii1\base\CApplication.php(180): CWebApplication->processRequest()
#2 D:\zyweb\pcb_logistics\website\home\index.php(38): CApplication->run()
#3 {main}
REQUEST_URI=/1
HTTP_REFERER=http://www.wllogistics.com/address/list
---
